<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Yuming Jiang" />
  <meta name="author" content="Quinn Dickinson" />
  <meta name="author" content="Amanda Momenzadeh" />
  <meta name="author" content="Jesse G. Meyer" />
  <meta name="dcterms.date" content="2022-01-05" />
  <meta name="keywords" content="proteomics, metabolomics, lipidomics, mass spectrometry" />
  <title>Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta name="citation_title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta property="og:title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta property="twitter:title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta name="dc.date" content="2022-01-05" />
  <meta name="citation_publication_date" content="2022-01-05" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Yuming Jiang" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_institution" content="Department of Whatever, University of Something" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="citation_author" content="Quinn Dickinson" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_institution" content="Department of Whatever, University of Something" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="citation_author" content="Amanda Momenzadeh" />
  <meta name="citation_author_institution" content="Department of Biochemistry, Medical College of Wisconsin" />
  <meta name="citation_author_orcid" content="0000-0002-8614-0690" />
  <meta name="citation_author" content="Jesse G. Meyer" />
  <meta name="citation_author_institution" content="Department of Biochemistry, Medical College of Wisconsin" />
  <meta name="citation_author_orcid" content="0000-0003-2753-3926" />
  <meta name="twitter:creator" content="@j_my_sci" />
  <link rel="canonical" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta property="og:url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta property="twitter:url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta name="citation_fulltext_html_url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta name="citation_pdf_url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e8fc1b23775a8ffad4a68720882a787098309887/" />
  <meta name="manubot_html_url_versioned" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e8fc1b23775a8ffad4a68720882a787098309887/" />
  <meta name="manubot_pdf_url_versioned" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e8fc1b23775a8ffad4a68720882a787098309887/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e8fc1b23775a8ffad4a68720882a787098309887/">permalink</a>)
was automatically generated
from <a href="https://github.com/jessegmeyerlab/2022-multi-omics-review/tree/e8fc1b23775a8ffad4a68720882a787098309887">jessegmeyerlab/2022-multi-omics-review@e8fc1b2</a>
on January 5, 2022.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Yuming Jiang</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/janeroe">janeroe</a><br>
<small>
Department of Something, University of Whatever; Department of Whatever, University of Something
</small></p></li>
<li><p><strong>Quinn Dickinson</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/janeroe">janeroe</a><br>
<small>
Department of Something, University of Whatever; Department of Whatever, University of Something
</small></p></li>
<li><p><strong>Amanda Momenzadeh</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8614-0690">0000-0002-8614-0690</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/amandamomenzadeh">amandamomenzadeh</a><br>
<small>
Department of Biochemistry, Medical College of Wisconsin
</small></p></li>
<li><p><strong>Jesse G. Meyer</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2753-3926">0000-0003-2753-3926</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jessegmeyerlab">jessegmeyerlab</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/j_my_sci">j_my_sci</a><br>
<small>
Department of Biochemistry, Medical College of Wisconsin
· Funded by Grant R21 AG074234; Grant R35 GM142502
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Studies that integrate unbiased measurements across at least two omics layers are often referred to as “multiomics”. Measurable “omes” include the genome, transcriptome, proteome, and metabolome. Any combination of omic measures can be referred to as multiomics; for the scope of this review we focus on research combining proteomics and metabolomics. Mass spectrometry is the leading technique for analysis of the proteome and the metabolome. Due to improvements in sample preparation and data collection, more studies are incorporating both mass spectrometry-based proteomics and metabolomics. In this review, we discuss the perceived value of multiomics, advances in sample preparation and data collection, the current state of multiomic data integration, and clinical examples of multiomic analysis. Finally, we explore major barriers preventing democratization of mass spectrometry based multiomics to the same level as nucleic acid analysis, and we suggest solutions to break these barriers.</p>
<h2 class="page_break_before" id="introduction">Introduction</h2>
<p>A major goal of biomedical research is understanding how changes in biomolecule compositions of cells and tissues lead to disease phenotypes.
The genome serves as a library of possible transcripts, some of which are instructions for proteins, and proteins act on metabolites.
The genome thus indirectly determines the set of possible cellular phenotypic states, and the exact cellular state at any given time depends on interaction between the genome with endogenous and exogenous environmental cues.
To understand how the genome connects to phenotype, measurement of the genome must be accompanied by measurement of downstream layers of the central dogma of biochemistry (Figure 1?).</p>
<p>Most RNAs encode for proteins, and aside from noncoding RNA and ribozymes, most of the potential functions encoded in a genome are carried out by proteins.
Although extensive research has nearly commoditized RNA measurment or transcriptomics <span class="citation" data-cites="yeKVZnPk">[<a href="#ref-yeKVZnPk" role="doc-biblioref">1</a>]</span>, multiple studies have found poor correlation between proteins and their corresponding mRNA <span class="citation" data-cites="12C1rMI72">[<a href="#ref-12C1rMI72" role="doc-biblioref">2</a>]</span>.
Therefore, functional understanding of a biological state demands quantitations of all proteins that are present.</p>
<p>Large scale study of the proteins in a biological system is known as proteomics <span class="citation" data-cites="K5BxV3br">[<a href="#ref-K5BxV3br" role="doc-biblioref">3</a>]</span>.
Proteomics originally meant large scale studies of protein quantities <span class="citation" data-cites="Xyl8WG4v">[<a href="#ref-Xyl8WG4v" role="doc-biblioref">4</a>]</span>, but it has grown to include large scale studies of protein structure<span class="citation" data-cites="xTQMmTt5">[<a href="#ref-xTQMmTt5" role="doc-biblioref">5</a>]</span> <span class="citation" data-cites="AC0VvwrU">[<a href="#ref-AC0VvwrU" role="doc-biblioref">6</a>]</span> <span class="citation" data-cites="IH1lj0uY">[<a href="#ref-IH1lj0uY" role="doc-biblioref">7</a>]</span>, protein-small molecule interactions<span class="citation" data-cites="1B1AiTt5z">[<a href="#ref-1B1AiTt5z" role="doc-biblioref">8</a>]</span>, and protein-protein interactions <span class="citation" data-cites="XD7JiQeK">[<a href="#ref-XD7JiQeK" role="doc-biblioref">9</a>]</span>..
Proteomics is increasingly applied to clinical studies to gain understanding of disease mechansims in humans <span class="citation" data-cites="mfHqlhNc">[<a href="#ref-mfHqlhNc" role="doc-biblioref">10</a>]</span>.</p>
<p>Many argue that quantities of proteins are not sufficient to enable understanding and dissection of biological systems.
The presence of a protein does not necessarily equal the presence of that protein’s function.
Protein functions are regulated in many ways, including allosteric feedback by metabolites and post-translational modifications such as phosphorylation, acylation, or proteolysis <span class="citation" data-cites="lumsFf8i">[<a href="#ref-lumsFf8i" role="doc-biblioref">11</a>]</span>.
Roughly 2,700 cellular proteins are predicted to be enzymes <span class="citation" data-cites="PBleB95P">[<a href="#ref-PBleB95P" role="doc-biblioref">12</a>]</span>, many of which act on metabolites that are intermediates required for cell proliferation or energy production.</p>
<p>The large scale measurement of the cellular metabolite pools is known as metabolomics <span class="citation" data-cites="NzLY5Rtq">[<a href="#ref-NzLY5Rtq" role="doc-biblioref">13</a>]</span>, which is sometimes further separated into polar metabolomics and lipidomics <span class="citation" data-cites="QdOZNn6Y">[<a href="#ref-QdOZNn6Y" role="doc-biblioref">14</a>]</span>.
Due to the chemical heterogeneity of metabolites, metabolomics studies use a variety of different separation and mass spectrometry (MS) techniques, including both gas chromatography (GC) <span class="citation" data-cites="tjwZSVe2">[<a href="#ref-tjwZSVe2" role="doc-biblioref">15</a>]</span> and liquid chromatograhpy (LC)<span class="citation" data-cites="WzQlzbjS">[<a href="#ref-WzQlzbjS" role="doc-biblioref">16</a>]</span>.
The field of metabolomics is trending toward more use of LC-MS over NMR <span class="citation" data-cites="OURTW6Q5">[<a href="#ref-OURTW6Q5" role="doc-biblioref">17</a>]</span>.</p>
<p>Measurement of multiple omic layers is know as multiomics.
MS measures peptides (or intact proteins <span class="citation" data-cites="a6Z8FNxW">[<a href="#ref-a6Z8FNxW" role="doc-biblioref">18</a>]</span>) for proteomic analysis, but MS also measures metabolites for metabolomic analysis.
Increasingly studies use MS to measure both proteomes and metabolomes from the same sample.
Although multiomics can refer to measuring at least any two omes, throughout this review we focus on multiomic measurements between proteomes and metabolomes.</p>
<p>Integrating data from multiomics should provide more information about the cellular state than the sum of each dataset.
Multiomic data integration methods have been developed with different classes of goals.
More work is needed to develop methods that take advantage of multiomic data to discover new biological insights into how systems work.</p>
<p>Previous reviews have covered related topics.
A recent review surveyed multiomic studies in mitochondria <span class="citation" data-cites="HnRPo54e">[<a href="#ref-HnRPo54e" role="doc-biblioref">19</a>]</span> with a focus on sample preparation, qualitity control, methods to collect MS data, and linking measured molecules to phenotypes of respirometry.
Another review gives an overview of each omic method, considerations before performing a multiomic method, and multiomic integration <span class="citation" data-cites="Vu9H5odK">[<a href="#ref-Vu9H5odK" role="doc-biblioref">20</a>]</span>.
Misra <em>et al.</em> reviewed multiomics with a focus on multiomic integration tools <span class="citation" data-cites="SPp3NcNL">[<a href="#ref-SPp3NcNL" role="doc-biblioref">21</a>]</span>.<br />
A metabolomics-centered review was recently published that discusses multiomic integration <span class="citation" data-cites="1FwHQW99U">[<a href="#ref-1FwHQW99U" role="doc-biblioref">22</a>]</span>.</p>
<p>In this review we focus on the following topics with a focus on MS-based multiomics studies that combine proteomics and metabolomics:</p>
<ul>
<li>sample prepartion methods</li>
<li>data collection methods</li>
<li>data integration</li>
<li>applications to model organisms</li>
<li>applications to clinical studies</li>
</ul>
<p>List of Planned Figures:</p>
<ol type="1">
<li>overview of how omic layers are related showing different ‘flavors’ of each omic analysis</li>
</ol>
<p>genomics: transcription factor binding (chip-seq), long range structure (ATAC seq?), etc
transcriptomics: microarrays, RNA-seq, long read nanopore or pacbio
proteomics: interactomics, structural proteomics (thermal proteome profiling or CETSA), PTMomics
metabolomics: polar metabolomics, lipidomics,</p>
<ol start="2" type="1">
<li><p>Workflow for typical MS-based multi-omics.
from sample preparation, extraction to data collection, data intergration and processing.</p></li>
<li><p>Table of clinical multiomics studies</p></li>
<li><p>Figure showing concepts in multiomics integration?</p></li>
<li></li>
</ol>
<h2 id="sample-preparation-for-multi-omic-analysis">Sample Preparation for Multi-Omic Analysis</h2>
<p>The basics: As with any methodology, the very first but also a crucial step for analytical success is proper sample preparation. This is particularly vital for mass spectrometry-based multi-omic analysis, the selected sample preparation strategy is a key determinant for information that will be obtained. Given the great diversity and complexity of biological samples being tested in proteomics and metabolomics, even subtle differences in sample preparation methods can have profound effects on the types of molecules being extracted, which further affect the retention time, signal stability and ionization efficiency. For example, analysis of different types of biomolecules requires specific sample extraction procedures. The most efficient extraction protocols often only cover a restricted type of biomolecules due to their different physicochemical properties.
After any necessary dissection or collection, biological tissue samples require a step to</p>
<p>1, Sample preparation for proteomics</p>
<p>Protein preparation for MS analysis can be accomplished by many methods, so it is important to understand the steps leading to analysis. While intact proteins are typically studied by gel electrophoresis, the most common mass spectrometry workflows for complex protein samples analyze peptides, which are easier than proteins to fractionate by LC. Peptides also ionize and fragment more efficiently than whole proteins, resulting in spectra that are easier to interpret for protein identification. Peptide preparation involves reduction and alkylation of cysteines, digestion of the sample into peptides, desalting and concentration of the peptides and final analysis of these peptides by ionization (e.g., ESI) plus orbitrap-based MS.</p>
<p>2, Sample preparation for metabolomics</p>
<p>2.1 non-targeted metabolomics</p>
<p><span class="citation" data-cites="Godx0lID">[<a href="#ref-Godx0lID" role="doc-biblioref">23</a>]</span></p>
<p>2.2 targeted metabolomics</p>
<p>2.3 lipidomics</p>
<p><span class="citation" data-cites="BfLOIXkt">[<a href="#ref-BfLOIXkt" role="doc-biblioref">24</a>]</span></p>
<p>3, Integrative sample prepatation for multi-omics</p>
<p>In the context of multi-omics analyses, being able to perform multiple measurements on the same sample can also decrease experimental variation. In this section,
we will review several integrative sample preparation technical advances that increased the capabilities of multiomic analysis. One important goal is to decrease the variation between different samples, ultimately to .</p>
<p><span class="citation" data-cites="10Oci3CsB">[<a href="#ref-10Oci3CsB" role="doc-biblioref">25</a>]</span></p>
<p><span class="citation" data-cites="IgbQlWYX">[<a href="#ref-IgbQlWYX" role="doc-biblioref">26</a>]</span></p>
<p>-“An Integrated, High-Throughput Strategy for Multiomic Systems Level Analysis” <span class="citation" data-cites="f4Sxxn1w">[<a href="#ref-f4Sxxn1w" role="doc-biblioref">27</a>]</span></p>
<p>-“Detergent-Free Simultaneous Sample Preparation Method for Proteomics and Metabolomics” <span class="citation" data-cites="6mBnEL4Y">[<a href="#ref-6mBnEL4Y" role="doc-biblioref">28</a>]</span></p>
<p>-“MPLEx: a Robust and Universal Protocol for Single-Sample Integrative Proteomic, Metabolomic, and Lipidomic Analyses” <span class="citation" data-cites="Mn37ufM0">[<a href="#ref-Mn37ufM0" role="doc-biblioref">29</a>]</span></p>
<p>-“Single-platform ‘multi-omic’ profiling: unified mass spectrometry and computational workflows for integrative proteomics–metabolomics analysis” <span class="citation" data-cites="YUQZsgXr">[<a href="#ref-YUQZsgXr" role="doc-biblioref">30</a>]</span></p>
<p>-“Three-in-One Simultaneous Extraction of Proteins, Metabolites and Lipids for Multi-Omics” <span class="citation" data-cites="18qG2TZbm">[<a href="#ref-18qG2TZbm" role="doc-biblioref">31</a>]</span></p>
<p>-”</p>
<ul>
<li></li>
</ul>
<h2 id="new-developments-of-mass-spectrometry-based-methods-for-multi-omics">New developments of mass spectrometry-based methods for multi-omics</h2>
<p>4.1 proteomics</p>
<p>4.1.1 Traditional standard methods for proteomics</p>
<p>(Remember to mention here)</p>
<p>4.1.2 Direct infusion methods for proteome analysis (high-throughput methods)</p>
<p>For current proteomic analysis methods, time-consuming chromatographic separation (typically requiring 30–60 min per sample or even longer) is required to protect the coverage,
repeatiability, robustness and quantification ability. However, with the rapid application of multiomics results in drug development, biomarker discovery studies and clinical
diagnosis. High-throughput methods is highly desirable to boost these fields forward.<span class="citation" data-cites="4ifoMVAQ">[<a href="#ref-4ifoMVAQ" role="doc-biblioref">32</a>]</span></p>
<p>(as a high-throughput method, MALDI based proteome analysis should be mentioned here, for example, the application of MALDI for identification of species of bacteria and fungas
through their specific peptides)</p>
<p>4.2 metabolomics</p>
<p>To accurately and reliably interpret data derived from metabolomics and lipidomics studies, enormous mass spectrometry based methods were developed during the past decades.
(remember to mention the application of MALDI for metabolites analysis, although the drawback of MALDI-tof is obvious.(</p>
<p>Drawbacks: 1,the background of organic matrix in the low molecular weight region
2,the obtained information of MALDI is still very limited, no more than 300 identified metabolites, and also quantification is difficult.<br />
3,as a non-consistent ion source, currently TOF is the typical mass analyser for MALDI, which still suffers from relative low resolution. FTICR can connect MALDI )</p>
<p>4.2.2 Direct infusion mass spectrometry methods for high-throughput analysis of metabolites.()</p>
<p>(direct infusion and so called flow injection MS. do not know the differences, seems saying the same thing.)
In typical metabolomic analysis workflow, MS is typically coupled with liquid or gas chromatography, enables separating metabolites within a complex sample and further increases the overall sensitivity by minimizing the ion suppression effect. However, this approach also bring for metabolomics analysis is that chromatographic separation is time-consuming (20–60 min per sample), preventing its application for high-throughput metabolomic screens of large scale samples that required for biomarker discovery studies and clinial analysis.</p>
<p><span class="citation" data-cites="BfLOIXkt">[<a href="#ref-BfLOIXkt" role="doc-biblioref">24</a>]</span></p>
<p><span class="citation" data-cites="BfLOIXkt">[<a href="#ref-BfLOIXkt" role="doc-biblioref">24</a>]</span></p>
<p>4.4 Integrated Strategy for Mass Spectrometry-Based Multiomics</p>
<p>Mass spectrometry (MS) serves as the centerpiece technology for proteome and metabolome analysis. To gain a better understanding of the multifaceted
networks of myriad actions in complex organisms, integration of different multiomic layers is increasingly explored such as joint methods of different omics.</p>
<p>“Multiomics Method Enabled by Sequential Metabolomics and Proteomics for Human Pluripotent Stem-Cell-Derived Cardiomyocytes” <span class="citation" data-cites="doi">[<a href="#ref-doi" role="doc-biblioref"><strong>doi?</strong></a>]</span></p>
<p><span class="citation" data-cites="15zlIdJ7">[<a href="#ref-15zlIdJ7" role="doc-biblioref">33</a>]</span></p>
<p><span class="citation" data-cites="13xdLvxHH">[<a href="#ref-13xdLvxHH" role="doc-biblioref">34</a>, =pdf]</span></p>
<p><span class="citation" data-cites="XTyNLxB5">[<a href="#ref-XTyNLxB5" role="doc-biblioref">35</a>, =pdf]</span></p>
<h2 id="multi-omic-data-integration">Multi-Omic Data Integration</h2>
<p>Argonaut data integration manuscript - <span class="citation" data-cites="YVw08vKm">[<a href="#ref-YVw08vKm" role="doc-biblioref">36</a>]</span></p>
<p>“Integration strategies of multi-omics data for machine learning analysis” <span class="citation" data-cites="FGRXe7Vp">[<a href="#ref-FGRXe7Vp" role="doc-biblioref">37</a>]</span></p>
<p>“Multi-omics data integration considerations and study design for biological systems and disease” <span class="citation" data-cites="8oXeZ6J4">[<a href="#ref-8oXeZ6J4" role="doc-biblioref">38</a>]</span></p>
<p>mixomics R tools <span class="citation" data-cites="pbomBYgV">[<a href="#ref-pbomBYgV" role="doc-biblioref">39</a>/journal.pcbi.1005752]</span></p>
<p>“Multi-omics Data Integration, Interpretation, and Its Application” <span class="citation" data-cites="15ZabGck2">[<a href="#ref-15ZabGck2" role="doc-biblioref">40</a>]</span></p>
<h2 id="mass-spectrometry-based-multi-omics-applied-to-model-organisms">Mass Spectrometry-based Multi-Omics Applied to Model Organisms</h2>
<p>“Multiomics Method Enabled by Sequential Metabolomics and Proteomics for Human Pluripotent Stem-Cell-Derived Cardiomyocytes” <span class="citation" data-cites="4LMKQFJ1">[<a href="#ref-4LMKQFJ1" role="doc-biblioref">41</a>]</span>:</p>
<p>Multi-omics Reveal Specific Targets of the RNA-Binding Protein Puf3p and Its Orchestration of Mitochondrial Biogenesis <span class="citation" data-cites="3SHP0O6L">[<a href="#ref-3SHP0O6L" role="doc-biblioref">42</a>]</span></p>
<p>Mitochondrial protein functions elucidated by multiomic mass spectrometry <span class="citation" data-cites="oTpHkkYw">[<a href="#ref-oTpHkkYw" role="doc-biblioref">43</a>]</span></p>
<p>Multi-omic mitoprotease profiling reveals role for oct1p <span class="citation" data-cites="mFITZcCh">[<a href="#ref-mFITZcCh" role="doc-biblioref">44</a>]</span></p>
<p>“An integrative systems genetic analysis of mammalian lipid metabolism” <span class="citation" data-cites="jTYumfe6">[<a href="#ref-jTYumfe6" role="doc-biblioref">45</a>]</span>
* proteomics and lipidomics of mouse liver across 107 genetically different strains
*</p>
<h2 id="clinical-applications-of-multi-omics-proteomics-metabolomics-andor-lipidomics">Clinical applications of multi-omics (proteomics, metabolomics, and/or lipidomics)</h2>
<p>Literature currently focused on proteogenomics (integration of genome, transcriptome, and proteome)“Clinical multi-omics strategies for the effective cancer management”<span class="citation" data-cites="HJCJR5cr">[<a href="#ref-HJCJR5cr" role="doc-biblioref">46</a>]</span></p>
<p>Holistic approach has the ability to improve prognostics and predictive accuracy of disease phenotypes to aid in better treatment and prevention
“Multi-omics Data Integration, Interpretation, and Its Application”<span class="citation" data-cites="1DLs9dWhH">[<a href="#ref-1DLs9dWhH" role="doc-biblioref">47</a>]</span></p>
<h2 id="approaches-in-cancer-research">Approaches in cancer research:</h2>
<ul>
<li>“Integration of Proteomics and Metabolomics Revealed Metabolite–Protein Networks in ACTH-Secreting Pituitary Adenoma” <span class="citation" data-cites="HJCJR5cr">[<a href="#ref-HJCJR5cr" role="doc-biblioref">46</a>]</span>: gas chromatography-mass spectrometry (GC-MS) for metabolomics, plus nano liquid chromatography tandem-mass spectrometry(nanoLC-MS/MS) proteomics. Using metabolomic and proteomic data to identify signaling pathways important in metabolic regulation of tumorigenesis. Reveal biomarkers for disease diagnosis, monitoring and therapeutic targets. Looking for changes in tumor on the pituitary gland (adrenocorticotropic hormone-secreting pituitary adenomas (ACTH-PA)). Downregulated glycolysis and fatty acid synthesis. Myc signaling pathway significantly participated in the metabolic changes and tumorigenesis of ACTH-PA.</li>
<li>“L-Arginine Modulates T Cell Metabolism and Enhances Survival and Anti-tumor Activity” <span class="citation" data-cites="k86tvAbl">[<a href="#ref-k86tvAbl" role="doc-biblioref">48</a>]</span>: investigation of proteomic and metabolic adaptations underlying CD4+ T cell activation using MS. Proteins were analyzed by liquid chromatography-coupled mass spectrometry (LC-MS). In parallel, polar metabolites were analyzed by non-targeted flow-injection metabolomics (semi-quantitative method that allows rapid and deep profiling of metabolites. limitations: isobaric compounds cannot be discriminated + possible in-source degradation). Findings: increased L-arginine levels has effects on T cell activation, differentiation, and function.</li>
<li>“Concurrent lipidomics and proteomics on malignant plasma cells from multiple myeloma patients: Probing the lipid metabolome” <span class="citation" data-cites="r7MXrdme">[<a href="#ref-r7MXrdme" role="doc-biblioref">49</a>]</span> comparing targeted and untargeted lipidomics and proteomics between relapsed and newly diagnosed MM patients</li>
<li>“Proteometabolomics of bladder cancer: Current and future prospects” <span class="citation" data-cites="doi:10.3233/CBM-150479">[<a href="#ref-doi:10.3233/CBM-150479" role="doc-biblioref"><strong>doi:10.3233/CBM-150479?</strong></a>]</span>: simultaneous proteome and metabolome data from urine and blood for urinary bladder cancer patient surveillance to aid in early detection of bladder cancer</li>
<li>“Grade-Dependent Metabolic Reprogramming in Kidney Cancer Revealed by Combined Proteomics and Metabolomics Analysis” <span class="citation" data-cites="17fdPwBLW">[<a href="#ref-17fdPwBLW" role="doc-biblioref">50</a>]</span></li>
<li>“Metabolomic/lipidomic profiling of COVID-19 and individual response to tocilizumab” https://doi.org/10.1371/journal.ppat.1009243</li>
<li>“Multi-Omics Resolves a Sharp Disease-State Shift between Mild and Moderate COVID-19” https://doi.org/10.1016/j.cell.2020.10.037</li>
<li>“Multi-Omics integration analysis of respiratory specimen characterizes baseline molecular determinants associated with COVID-19 diagnosis” https://doi.org/10.1101/2020.07.06.20147082</li>
<li>“Longitudinal multi-omics analysis identifies responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19 trajectories” https://doi.org/10.1101/2020.09.11.20187369</li>
</ul>
<h2 id="approaches-in-covid-19-research">Approaches in COVID-19 research:</h2>
<ul>
<li>“Proteomic and Metabolomic Characterization of COVID-19 Patient Sera” <span class="citation" data-cites="VwgXtxp7">[<a href="#ref-VwgXtxp7" role="doc-biblioref">51</a>]</span>: proteomic and metabolomic profiling of serum from 46 COVID-19 patients. Stable isotope labeled proteomics TMTpro (16plex) and and ultra performance liquid chromatography/tandem mass spectrometry (UPLC-MS/MS) untargeted metabolomics to analyze samples. Dysregulation of macrophage, platelet degranulation, complement system, and metabolic suppression in severe COVID-19 patients, useful for selection of biomarkers for severity evaluation. Implemented machine learning model using expression levels of 22 serum proteins and 7 metabolites with overall accuracy of 93.5% in the training set.
*Jaitovich paper: <span class="citation" data-cites="zktsqjwc">[<a href="#ref-zktsqjwc" role="doc-biblioref">52</a>]</span>: investigation of proteins, lipids, and metabolites associated with COVID-19 status and severity in pts admitted with moderate to severe respiratory issues presumably related to SARS-CoV-2 infection.</li>
</ul>
<h2 id="misc-disease-states">Misc disease states:</h2>
<ul>
<li>“Integration of metabolomics and proteomics in multiple sclerosis: From biomarkers discovery to personalized medicine” <span class="citation" data-cites="ckNm0Nmf">[<a href="#ref-ckNm0Nmf" role="doc-biblioref">53</a>]</span>: majority of biomarkers for MS are not sensitive and/or specific enough to be used for population screening. From the 188 proposed biomarker candidates for MS found in CSF, only 10 (5%) have been successfully verified, while 20 have been falsified. This is likely due to the inter-individual variation. Integrating omics approaches may help with this pitfall; proteomics in combination with the other “omics” may synergize to proide a more precise spectrum of information capable of better characterizing a phenotype. When proteins alone, metabolites alone and proteins and metabolites together were uploaded into the ingenuity pathway analysis (IPA) tool, number of matched molecules for “proteins and metabolites” is higher than the sum of molecules matched in the separate searches.</li>
<li>“High-throughput mediation analysis of human proteome and metabolome identifies mediators of post-bariatric surgical diabetes control”<span class="citation" data-cites="kj9iEc2i">[<a href="#ref-kj9iEc2i" role="doc-biblioref">54</a>]</span> patients with type 2 diabetes randomized to RYGB vs nonsurgical diabetes/weight management; fasting plasma proteome and metabolome were assayed up to 3 years. Plasma proteome profiling was performed using the high-throughput DNA aptamer-based SOMAscan assay platform. Plasma metabolomics were profiled using a commercial semi-quantitative mass spectrometry-based platform (Metabolon, Inc.) using UPLC-MS/MS. Main mediator of improved glycemic control was GHR, which was reduced 3 months after RYGB. GH signaling = diabetogenic effects.</li>
</ul>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-yeKVZnPk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>RNA-Seq: a revolutionary tool for transcriptomics</strong> <div class="csl-block">Zhong Wang, Mark Gerstein, Michael Snyder</div> <em>Nature Reviews Genetics</em> (2009-01) <a href="https://doi.org/crrz3v">https://doi.org/crrz3v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nrg2484">https://doi.org/10.1038/nrg2484</a></div></div>
</div>
<div id="ref-12C1rMI72" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Insights into the regulation of protein abundance from proteomic and transcriptomic analyses</strong> <div class="csl-block">Christine Vogel, Edward M Marcotte</div> <em>Nature Reviews Genetics</em> (2012-04) <a href="https://doi.org/bg9g">https://doi.org/bg9g</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nrg3185">https://doi.org/10.1038/nrg3185</a></div></div>
</div>
<div id="ref-K5BxV3br" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>Mass-spectrometric exploration of proteome structure and function</strong> <div class="csl-block">Ruedi Aebersold, Matthias Mann</div> <em>Nature</em> (2016-09) <a href="https://doi.org/f83zqm">https://doi.org/f83zqm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature19949">https://doi.org/10.1038/nature19949</a></div></div>
</div>
<div id="ref-Xyl8WG4v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Quantitative proteomics: challenges and opportunities in basic and applied research</strong> <div class="csl-block">Olga T Schubert, Hannes L Röst, Ben C Collins, George Rosenberger, Ruedi Aebersold</div> <em>Nature Protocols</em> (2017-07) <a href="https://doi.org/gbhfhm">https://doi.org/gbhfhm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2017.040">10.1038/nprot.2017.040</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28569762">28569762</a></div></div>
</div>
<div id="ref-xTQMmTt5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>Measuring protein structural changes on a proteome-wide scale using limited proteolysis-coupled mass spectrometry</strong> <div class="csl-block">Simone Schopper, Abdullah Kahraman, Pascal Leuenberger, Yuehan Feng, Ilaria Piazza, Oliver Müller, Paul J Boersema, Paola Picotti</div> <em>Nature Protocols</em> (2017-11) <a href="https://doi.org/gcf7zn">https://doi.org/gcf7zn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2017.100">10.1038/nprot.2017.100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29072706">29072706</a></div></div>
</div>
<div id="ref-AC0VvwrU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Unlocking the secrets to protein–protein interface drug targets using structural mass spectrometry techniques</strong> <div class="csl-block">Angela Dailing, Alessandra Luchini, Lance Liotta</div> <em>Expert Review of Proteomics</em> (2015-09-03) <a href="https://doi.org/f74qnh">https://doi.org/f74qnh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/14789450.2015.1079487">10.1586/14789450.2015.1079487</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26400464">26400464</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4858316">PMC4858316</a></div></div>
</div>
<div id="ref-IH1lj0uY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Fast photochemical oxidation of proteins (FPOP): A powerful mass spectrometry–based structural proteomics tool</strong> <div class="csl-block">Danté T Johnson, Luciano H Di Stefano, Lisa M Jones</div> <em>Journal of Biological Chemistry</em> (2019-08) <a href="https://doi.org/ghzz8z">https://doi.org/ghzz8z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/jbc.rev119.006218">10.1074/jbc.rev119.006218</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31262727">31262727</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6690683">PMC6690683</a></div></div>
</div>
<div id="ref-1B1AiTt5z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Drug Target Identification in Tissues by Thermal Proteome Profiling</strong> <div class="csl-block">André Mateus, Nils Kurzawa, Jessica Perrin, Giovanna Bergamini, Mikhail M Savitski</div> <em>Annual Review of Pharmacology and Toxicology</em> (2022-01-06) <a href="https://doi.org/gmwmqn">https://doi.org/gmwmqn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev-pharmtox-052120-013205">10.1146/annurev-pharmtox-052120-013205</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34499524">34499524</a></div></div>
</div>
<div id="ref-XD7JiQeK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><a href="https://doi.org/10.1016/j.coisb.2017.04.015">https://doi.org/10.1016/j.coisb.2017.04.015</a></div>
</div>
<div id="ref-mfHqlhNc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Clinical applications of quantitative proteomics using targeted and untargeted data-independent acquisition techniques.</strong> <div class="csl-block">Jesse G Meyer, Birgit Schilling</div> <em>Expert review of proteomics</em> (2017-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28436239">https://www.ncbi.nlm.nih.gov/pubmed/28436239</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1080/14789450.2017.1322904">10.1080/14789450.2017.1322904</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28436239">28436239</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5671767">PMC5671767</a></div></div>
</div>
<div id="ref-lumsFf8i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Post-translational modification: nature's escape from genetic imprisonment and the basis for dynamic information encoding: Information encoding by post-translational modification</strong> <div class="csl-block">Sudhakaran Prabakaran, Guy Lippens, Hanno Steen, Jeremy Gunawardena</div> <em>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</em> (2012-11) <a href="https://doi.org/f39h4k">https://doi.org/f39h4k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/wsbm.1185">10.1002/wsbm.1185</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22899623">22899623</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3473174">PMC3473174</a></div></div>
</div>
<div id="ref-PBleB95P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>[No title found]</strong> <div class="csl-block">Pedro Romero, Jonathan Wagg, Michelle L Green, Dale Kaiser, Markus Krummenacker, Peter D Karp</div> <em>Genome Biology</em> (2004) <a href="https://doi.org/fmkcgb">https://doi.org/fmkcgb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/gb-2004-6-1-r2">10.1186/gb-2004-6-1-r2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15642094">15642094</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC549063">PMC549063</a></div></div>
</div>
<div id="ref-NzLY5Rtq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>Mass spectrometry-based metabolomics in health and medical science: a systematic review</strong> <div class="csl-block">Xi-wu Zhang, Qiu-han Li, Zuo-di Xu, Jin-jin Dou</div> <em>RSC Advances</em> (2020) <a href="https://doi.org/gnxx6f">https://doi.org/gnxx6f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c9ra08985c">https://doi.org/10.1039/c9ra08985c</a></div></div>
</div>
<div id="ref-QdOZNn6Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>Mass spectrometry-based shotgun lipidomics – a critical review from the technical point of view</strong> <div class="csl-block">Fong-Fu Hsu</div> <em>Analytical and Bioanalytical Chemistry</em> (2018-10) <a href="https://doi.org/gd86z9">https://doi.org/gd86z9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00216-018-1252-y">https://doi.org/10.1007/s00216-018-1252-y</a></div></div>
</div>
<div id="ref-tjwZSVe2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Metabolomics by Gas Chromatography-Mass Spectrometry: Combined Targeted and Untargeted Profiling.</strong> <div class="csl-block">Oliver Fiehn</div> <em>Current protocols in molecular biology</em> (2016-04-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27038389">https://www.ncbi.nlm.nih.gov/pubmed/27038389</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/0471142727.mb3004s114">10.1002/0471142727.mb3004s114</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27038389">27038389</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4829120">PMC4829120</a></div></div>
</div>
<div id="ref-WzQlzbjS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Separation strategies for untargeted metabolomics</strong> <div class="csl-block">Gary J Patti</div> <em>Journal of Separation Science</em> (2011-12) <a href="https://doi.org/b5zd3w">https://doi.org/b5zd3w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jssc.201100532">10.1002/jssc.201100532</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21972197">21972197</a></div></div>
</div>
<div id="ref-OURTW6Q5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Novel technologies for metabolomics: More for less</strong> <div class="csl-block">Paul Miggiels, Bert Wouters, Gerard JP van Westen, Anne-Charlotte Dubbelman, Thomas Hankemeier</div> <em>TrAC Trends in Analytical Chemistry</em> (2019-11) <a href="https://doi.org/gnx98m">https://doi.org/gnx98m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.trac.2018.11.021">10.1016/j.trac.2018.11.021</a></div></div>
</div>
<div id="ref-a6Z8FNxW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>High-throughput quantitative top-down proteomics</strong> <div class="csl-block">Kellye A Cupp-Sutton, Si Wu</div> <em>Molecular Omics</em> (2020) <a href="https://doi.org/gnx98p">https://doi.org/gnx98p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c9mo00154a">10.1039/c9mo00154a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31932818">31932818</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7529119">PMC7529119</a></div></div>
</div>
<div id="ref-HnRPo54e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Mass-spectrometric multi-omics linked to function – State-of-the-art investigations of mitochondria in systems medicine</strong> <div class="csl-block">TrAC Trends in Analytical Chemistry</div> (2019-10-01) <a href="https://www.sciencedirect.com/science/article/pii/S0165993619303668">https://www.sciencedirect.com/science/article/pii/S0165993619303668</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.trac.2019.115635">10.1016/j.trac.2019.115635</a></div></div>
</div>
<div id="ref-Vu9H5odK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Multi-omics approaches to disease</strong> <div class="csl-block">Yehudit Hasin, Marcus Seldin, Aldons Lusis</div> <em>Genome Biology</em> (2017-05-05) <a href="https://doi.org/10.1186/s13059-017-1215-1">https://doi.org/10.1186/s13059-017-1215-1</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s13059-017-1215-1">10.1186/s13059-017-1215-1</a></div></div>
</div>
<div id="ref-SPp3NcNL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Integrated omics: tools, advances and future approaches</strong> <div class="csl-block">Biswapriya B Misra, Carl Langefeld, Michael Olivier, Laura A Cox</div> <em>Journal of Molecular Endocrinology</em> (2019-01) <a href="https://doi.org/gfx8hg">https://doi.org/gfx8hg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1530/jme-18-0055">https://doi.org/10.1530/jme-18-0055</a></div></div>
</div>
<div id="ref-1FwHQW99U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Multi-omics integration in biomedical research – A metabolomics-centric review</strong> <div class="csl-block">Maria A Wörheide, Jan Krumsiek, Gabi Kastenmüller, Matthias Arnold</div> <em>Analytica Chimica Acta</em> (2021-01) <a href="https://doi.org/gmsz75">https://doi.org/gmsz75</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2020.10.038">https://doi.org/10.1016/j.aca.2020.10.038</a></div></div>
</div>
<div id="ref-Godx0lID" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Development of a plasma pseudotargeted metabolomics method based on ultra-high-performance liquid chromatography–mass spectrometry</strong> <div class="csl-block">Fujian Zheng, Xinjie Zhao, Zhongda Zeng, Lichao Wang, Wangjie Lv, Qingqing Wang, Guowang Xu</div> <em>Nature Protocols</em> (2020-08) <a href="https://www.nature.com/articles/s41596-020-0341-5">https://www.nature.com/articles/s41596-020-0341-5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41596-020-0341-5">10.1038/s41596-020-0341-5</a></div></div>
</div>
<div id="ref-BfLOIXkt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>A complete workflow for high-resolution spectral-stitching nanoelectrospray direct-infusion mass-spectrometry-based metabolomics and lipidomics</strong> <div class="csl-block">Andrew D Southam, Ralf JM Weber, Jasper Engel, Martin R Jones, Mark R Viant</div> <em>Nature Protocols</em> (2017-02) <a href="https://www.nature.com/articles/nprot.2016.156">https://www.nature.com/articles/nprot.2016.156</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2016.156">10.1038/nprot.2016.156</a></div></div>
</div>
<div id="ref-10Oci3CsB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Multiomic analysis of a dried single-drop plasma sample using an integrated mass spectrometry approach</strong> <div class="csl-block">Weina Gao, Qiaoyun Zhang, Yiran Su, Peiwu Huang, Xue Lu, Qinyue Gong, Wendong Chen, Ruilian Xu, Ruijun Tian</div> <em>Analyst</em> (2020-10-12) <a href="https://pubs.rsc.org/en/content/articlelanding/2020/an/d0an01149e">https://pubs.rsc.org/en/content/articlelanding/2020/an/d0an01149e</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/D0AN01149E">10.1039/d0an01149e</a></div></div>
</div>
<div id="ref-IgbQlWYX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><a href="https://doi.org/10.3389/fgene.2021.635971">https://doi.org/10.3389/fgene.2021.635971</a></div>
</div>
<div id="ref-f4Sxxn1w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>An Integrated, High-Throughput Strategy for Multiomic Systems Level Analysis</strong> <div class="csl-block">Danielle B Gutierrez, Randi L Gant-Branum, Carrie E Romer, Melissa A Farrow, Jamie L Allen, Nikesh Dahal, Yuan-Wei Nei, Simona G Codreanu, Ashley T Jordan, Lauren D Palmer, … Richard M Caprioli</div> <em>Journal of Proteome Research</em> (2018-10-05) <a href="https://doi.org/gfhjtv">https://doi.org/gfhjtv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00302">10.1021/acs.jproteome.8b00302</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30114907">30114907</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6469931">PMC6469931</a></div></div>
</div>
<div id="ref-6mBnEL4Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>Detergent-Free Simultaneous Sample Preparation Method for Proteomics and Metabolomics</strong> <div class="csl-block">Alexandre Zougman, John P Wilson, Lee D Roberts, Rosamonde E Banks</div> <em>Journal of Proteome Research</em> (2020-07-02) <a href="https://doi.org/gnxfr9">https://doi.org/gnxfr9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.9b00662">https://doi.org/10.1021/acs.jproteome.9b00662</a></div></div>
</div>
<div id="ref-Mn37ufM0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>MPLEx: a Robust and Universal Protocol for Single-Sample Integrative Proteomic, Metabolomic, and Lipidomic Analyses</strong> <div class="csl-block">Ernesto S Nakayasu, Carrie D Nicora, Amy C Sims, Kristin E Burnum-Johnson, Young-Mo Kim, Jennifer E Kyle, Melissa M Matzke, Anil K Shukla, Rosalie K Chu, Athena A Schepmoes, … Thomas O Metz</div> <em>mSystems</em> (2016-06-28) <a href="https://doi.org/gk7gvg">https://doi.org/gk7gvg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1128/msystems.00043-16">https://doi.org/10.1128/msystems.00043-16</a></div></div>
</div>
<div id="ref-YUQZsgXr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>Single-platform ‘multi-omic’ profiling: unified mass spectrometry and computational workflows for integrative proteomics–metabolomics analysis</strong> <div class="csl-block">Benjamin C Blum, Fatemeh Mousavi, Andrew Emili</div> <em>Molecular Omics</em> (2018) <a href="https://doi.org/ghcv4s">https://doi.org/ghcv4s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c8mo00136g">10.1039/c8mo00136g</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30211418">30211418</a></div></div>
</div>
<div id="ref-18qG2TZbm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Three-in-One Simultaneous Extraction of Proteins, Metabolites and Lipids for Multi-Omics</strong> <div class="csl-block">Jianing Kang, Lisa David, Yangyang Li, Jing Cang, Sixue Chen</div> <em>Frontiers in Genetics</em> (2021-04-15) <a href="https://doi.org/gnxfsb">https://doi.org/gnxfsb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fgene.2021.635971">https://doi.org/10.3389/fgene.2021.635971</a></div></div>
</div>
<div id="ref-4ifoMVAQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>Quantitative shotgun proteome analysis by direct infusion</strong> <div class="csl-block">Jesse G Meyer, Natalie M Niemi, David J Pagliarini, Joshua J Coon</div> <em>Nature Methods</em> (2020-12) <a href="https://www.nature.com/articles/s41592-020-00999-z">https://www.nature.com/articles/s41592-020-00999-z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-020-00999-z">10.1038/s41592-020-00999-z</a></div></div>
</div>
<div id="ref-15zlIdJ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>An Integrated Strategy for Mass Spectrometry-Based Multiomics Analysis of Single Cells</strong> <div class="csl-block">Yuanyuan Li, Hang Li, Yuping Xie, Shuo Chen, Ritian Qin, Hangyan Dong, Yongliang Yu, Jianhua Wang, Xiaohong Qian, Weijie Qin</div> <em>Analytical Chemistry</em> (2021-10-13) <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c05209">https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c05209</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c05209">10.1021/acs.analchem.0c05209</a></div></div>
</div>
<div id="ref-13xdLvxHH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Multi-Omic Single-Shot Technology for Integrated Proteome and Lipidome Analysis</strong> <div class="csl-block">Yuchen He, Edrees H Rashan, Vanessa Linke, Evgenia Shishkova, Alexander S Hebert, Adam Jochem, Michael S Westphall, David J Pagliarini, Katherine A Overmyer, Joshua J Coon</div> <em>Analytical Chemistry</em> (2021-02-22) <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c04764">https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c04764</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c04764">10.1021/acs.analchem.0c04764</a></div></div>
</div>
<div id="ref-XTyNLxB5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>An Integrated Strategy for Mass Spectrometry-Based Multiomics Analysis of Single Cells</strong> <div class="csl-block">Yuanyuan Li, Hang Li, Yuping Xie, Shuo Chen, Ritian Qin, Hangyan Dong, Yongliang Yu, Jianhua Wang, Xiaohong Qian, Weijie Qin</div> <em>Analytical Chemistry</em> (2021-10-13) <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c05209">https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c05209</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c05209">10.1021/acs.analchem.0c05209</a></div></div>
</div>
<div id="ref-YVw08vKm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><a href="https://doi.org/10.1016/j.patter.2020.100122">https://doi.org/10.1016/j.patter.2020.100122</a></div>
</div>
<div id="ref-FGRXe7Vp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Integration strategies of multi-omics data for machine learning analysis.</strong> <div class="csl-block">Milan Picard, Marie-Pier Scott-Boyer, Antoine Bodein, Olivier Périn, Arnaud Droit</div> <em>Computational and structural biotechnology journal</em> (2021-06-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34285775">https://www.ncbi.nlm.nih.gov/pubmed/34285775</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.csbj.2021.06.030">10.1016/j.csbj.2021.06.030</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34285775">34285775</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8258788">PMC8258788</a></div></div>
</div>
<div id="ref-8oXeZ6J4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>Multi-omics data integration considerations and study design for biological systems and disease.</strong> <div class="csl-block">Stefan Graw, Kevin Chappell, Charity L Washam, Allen Gies, Jordan Bird, Michael S Robeson, Stephanie D Byrum</div> <em>Molecular omics</em> (2021-04-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33347526">https://www.ncbi.nlm.nih.gov/pubmed/33347526</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/d0mo00041h">10.1039/d0mo00041h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33347526">33347526</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8058243">PMC8058243</a></div></div>
</div>
<div id="ref-pbomBYgV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><a href="https://journals.plos.org/ploscompbiol/article?id">https://journals.plos.org/ploscompbiol/article?id</a></div>
</div>
<div id="ref-15ZabGck2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>Multi-omics Data Integration, Interpretation, and Its Application</strong> <div class="csl-block">Indhupriya Subramanian, Srikant Verma, Shiva Kumar, Abhay Jere, Krishanpal Anamika</div> <em>Bioinformatics and Biology Insights</em> (2020-01) <a href="https://doi.org/ggn3c8">https://doi.org/ggn3c8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1177/1177932219899051">https://doi.org/10.1177/1177932219899051</a></div></div>
</div>
<div id="ref-4LMKQFJ1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Multiomics Method Enabled by Sequential Metabolomics and Proteomics for Human Pluripotent Stem-Cell-Derived Cardiomyocytes</strong> <div class="csl-block">Elizabeth F Bayne, Aaron D Simmons, David S Roberts, Yanlong Zhu, Timothy J Aballo, Benjamin Wancewicz, Sean P Palecek, Ying Ge</div> <em>Journal of Proteome Research</em> (2021-10-01) <a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.1c00611">https://pubs.acs.org/doi/10.1021/acs.jproteome.1c00611</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00611">10.1021/acs.jproteome.1c00611</a></div></div>
</div>
<div id="ref-3SHP0O6L" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>Multi-omics Reveal Specific Targets of the RNA-Binding Protein Puf3p and Its Orchestration of Mitochondrial Biogenesis.</strong> <div class="csl-block">Christopher P Lapointe, Jonathan A Stefely, Adam Jochem, Paul D Hutchins, Gary M Wilson, Nicholas W Kwiecien, Joshua J Coon, Marvin Wickens, David J Pagliarini</div> <em>Cell systems</em> (2017-12-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29248374">https://www.ncbi.nlm.nih.gov/pubmed/29248374</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cels.2017.11.012">10.1016/j.cels.2017.11.012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29248374">29248374</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5799006">PMC5799006</a></div></div>
</div>
<div id="ref-oTpHkkYw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>Mitochondrial protein functions elucidated by multi-omic mass spectrometry profiling.</strong> <div class="csl-block">Jonathan A Stefely, Nicholas W Kwiecien, Elyse C Freiberger, Alicia L Richards, Adam Jochem, Matthew JP Rush, Arne Ulbrich, Kyle P Robinson, Paul D Hutchins, Mike T Veling, … Joshua J Coon</div> <em>Nature biotechnology</em> (2016-09-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27669165">https://www.ncbi.nlm.nih.gov/pubmed/27669165</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.3683">10.1038/nbt.3683</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27669165">27669165</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5101133">PMC5101133</a></div></div>
</div>
<div id="ref-mFITZcCh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>Multi-omic Mitoprotease Profiling Defines a Role for Oct1p in Coenzyme Q Production.</strong> <div class="csl-block">Mike T Veling, Andrew G Reidenbach, Elyse C Freiberger, Nicholas W Kwiecien, Paul D Hutchins, Michael J Drahnak, Adam Jochem, Arne Ulbrich, Matthew JP Rush, Jason D Russell, … David J Pagliarini</div> <em>Molecular cell</em> (2017-12-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29220658">https://www.ncbi.nlm.nih.gov/pubmed/29220658</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.molcel.2017.11.023">10.1016/j.molcel.2017.11.023</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29220658">29220658</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5730362">PMC5730362</a></div></div>
</div>
<div id="ref-jTYumfe6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>An integrative systems genetic analysis of mammalian lipid metabolism.</strong> <div class="csl-block">Benjamin L Parker, Anna C Calkin, Marcus M Seldin, Michael F Keating, Elizabeth J Tarling, Pengyi Yang, Sarah C Moody, Yingying Liu, Eser J Zerenturk, Elise J Needham, … Brian G Drew</div> <em>Nature</em> (2019-02-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30814737">https://www.ncbi.nlm.nih.gov/pubmed/30814737</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-019-0984-y">10.1038/s41586-019-0984-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30814737">30814737</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6656374">PMC6656374</a></div></div>
</div>
<div id="ref-HJCJR5cr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline"><a href="https://doi.org/10.3389/fendo.2018.00678">https://doi.org/10.3389/fendo.2018.00678</a></div>
</div>
<div id="ref-1DLs9dWhH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline"><strong>Multi-omics Data Integration, Interpretation, and Its Application:</strong> <div class="csl-block">Indhupriya Subramanian, Srikant Verma, Shiva Kumar, Abhay Jere, Krishanpal Anamika</div> <em>Bioinformatics and Biology Insights</em> (2020-01-31) <a href="https://journals.sagepub.com/doi/10.1177/1177932219899051">https://journals.sagepub.com/doi/10.1177/1177932219899051</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1177/1177932219899051">10.1177/1177932219899051</a></div></div>
</div>
<div id="ref-k86tvAbl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline"><strong>L-Arginine Modulates T Cell Metabolism and Enhances Survival and Anti-tumor Activity</strong> <div class="csl-block">Roger Geiger, Jan C Rieckmann, Tobias Wolf, Camilla Basso, Yuehan Feng, Tobias Fuhrer, Maria Kogadeeva, Paola Picotti, Felix Meissner, Matthias Mann, … Antonio Lanzavecchia</div> <em>Cell</em> (2016-10) <a href="https://doi.org/f88qkm">https://doi.org/f88qkm</a> <div class="csl-block">DOI: <a href="http://dx.doi.org/10.1016/j.cell.2016.09.031">http://dx.doi.org/10.1016/j.cell.2016.09.031</a></div></div>
</div>
<div id="ref-r7MXrdme" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline"><strong>Concurrent lipidomics and proteomics on malignant plasma cells from multiple myeloma patients: Probing the lipid metabolome</strong> <div class="csl-block">Ahmed Mohamed, Joel Collins, Hui Jiang, Jeffrey Molendijk, Thomas Stoll, Federico Torta, Markus R Wenk, Robert J Bird, Paula Marlton, Peter Mollee, … Michelle M Hill</div> <em>PLOS ONE</em> (2020-01-08) <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0227455">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0227455</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0227455">10.1371/journal.pone.0227455</a></div></div>
</div>
<div id="ref-17fdPwBLW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline"><strong>Grade-Dependent Metabolic Reprogramming in Kidney Cancer Revealed by Combined Proteomics and Metabolomics Analysis</strong> <div class="csl-block">Hiromi I Wettersten, AAri Hakimi, Dexter Morin, Cristina Bianchi, Megan E Johnstone, Dallas R Donohoe, Josephine F Trott, Omran Abu Aboud, Steven Stirdivant, Bruce Neri, … Robert H Weiss</div> <em>Cancer Research</em> (2015-06-15) <a href="https://doi.org/f7h8zh">https://doi.org/f7h8zh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1158/0008-5472.can-14-1703">10.1158/0008-5472.can-14-1703</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25952651">25952651</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4470795">PMC4470795</a></div></div>
</div>
<div id="ref-VwgXtxp7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline"><a href="https://doi.org/10.1016/j.cell.2020.05.032">https://doi.org/10.1016/j.cell.2020.05.032</a></div>
</div>
<div id="ref-zktsqjwc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline"><a href="https://doi.org/10.1016/j.cels.2020.10.003">https://doi.org/10.1016/j.cels.2020.10.003</a></div>
</div>
<div id="ref-ckNm0Nmf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline"><strong>Integration of metabolomics and proteomics in multiple sclerosis: From biomarkers discovery to personalized medicine</strong> <div class="csl-block">Piero Del Boccio, Claudia Rossi, Maria di Ioia, Ilaria Cicalini, Paolo Sacchetta, Damiana Pieragostino</div> <em>PROTEOMICS – Clinical Applications</em> (2016-04-01) <a href="https://onlinelibrary.wiley.com/doi/10.1002/prca.201500083">https://onlinelibrary.wiley.com/doi/10.1002/prca.201500083</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/prca.201500083">10.1002/prca.201500083</a></div></div>
</div>
<div id="ref-kj9iEc2i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline"><strong>High-throughput mediation analysis of human proteome and metabolome identifies mediators of post-bariatric surgical diabetes control</strong> <div class="csl-block">Jonathan M Dreyfuss, Yixing Yuchi, Xuehong Dong, Vissarion Efthymiou, Hui Pan, Donald C Simonson, Ashley Vernon, Florencia Halperin, Pratik Aryal, Anish Konkar, … Mary Elizabeth Patti</div> <em>Nature Communications</em> (2021-11-29) <a href="https://www.nature.com/articles/s41467-021-27289-2">https://www.nature.com/articles/s41467-021-27289-2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-021-27289-2">10.1038/s41467-021-27289-2</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
