---
title: 'Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics'
keywords:
- proteomics
- metabolomics
- lipidomics
- mass spectrometry
lang: en-US
date-meta: '2021-12-28'
author-meta:
- Yuming Jiang
- Quinn Dickinson
- Amanda Momenzadeh
- Jesse G. Meyer
header-includes: |-
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta name="citation_title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta property="og:title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta property="twitter:title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta name="dc.date" content="2021-12-28" />
  <meta name="citation_publication_date" content="2021-12-28" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Yuming Jiang" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_institution" content="Department of Whatever, University of Something" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="citation_author" content="Quinn Dickinson" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_institution" content="Department of Whatever, University of Something" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="citation_author" content="Amanda Momenzadeh" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_institution" content="Department of Whatever, University of Something" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="citation_author" content="Jesse G. Meyer" />
  <meta name="citation_author_institution" content="Department of Biochemistry, Medical College of Wisconsin" />
  <meta name="citation_author_orcid" content="0000-0003-2753-3926" />
  <meta name="twitter:creator" content="@j_my_sci" />
  <link rel="canonical" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta property="og:url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta property="twitter:url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta name="citation_fulltext_html_url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta name="citation_pdf_url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/da88a4a4ba01a8b05e2c0ecb3c60d0d7ff400338/" />
  <meta name="manubot_html_url_versioned" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/da88a4a4ba01a8b05e2c0ecb3c60d0d7ff400338/" />
  <meta name="manubot_pdf_url_versioned" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/da88a4a4ba01a8b05e2c0ecb3c60d0d7ff400338/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
bibliography:
- content/manual-references.json
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
manubot-clear-requests-cache: false
...






<small><em>
This manuscript
([permalink](https://jessegmeyerlab.github.io/2022-multi-omics-review/v/da88a4a4ba01a8b05e2c0ecb3c60d0d7ff400338/))
was automatically generated
from [jessegmeyerlab/2022-multi-omics-review@da88a4a](https://github.com/jessegmeyerlab/2022-multi-omics-review/tree/da88a4a4ba01a8b05e2c0ecb3c60d0d7ff400338)
on December 28, 2021.
</em></small>

## Authors



+ **Yuming Jiang**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [janeroe](https://github.com/janeroe)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
  </small>

+ **Quinn Dickinson**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [janeroe](https://github.com/janeroe)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
  </small>

+ **Amanda Momenzadeh**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [janeroe](https://github.com/janeroe)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
  </small>

+ **Jesse G. Meyer**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [0000-0003-2753-3926](https://orcid.org/0000-0003-2753-3926)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [jessegmeyerlab](https://github.com/jessegmeyerlab)
    · ![Twitter icon](images/twitter.svg){.inline_icon width=16 height=16}
    [j_my_sci](https://twitter.com/j_my_sci)<br>
  <small>
     Department of Biochemistry, Medical College of Wisconsin
     · Funded by Grant R21 AG074234; Grant R35 GM142502
  </small>



## Abstract {.page_break_before}

Studies that integrate unbiased measurements across at least two omics layers are often referred to as "multiomics". Measurable "omes" include the genome, transcriptome, proteome, and metabolome. Any combination of omic measures can be referred to as multiomics; for the scope of this review we focus on research combining proteomics and metabolomics. Mass spectrometry is the leading technique for analysis of the proteome and the metabolome. Due to improvements in sample preparation and data collection, more studies are incorporating both mass spectrometry-based proteomics and metabolomics. In this review, we discuss the perceived value of multiomics, advances in sample preparation and data collection, the current state of multiomic data integration, and clinical examples of multiomic analysis. Finally, we explore major barriers preventing democratization of mass spectrometry based multiomics to the same level as nucleic acid analysis, and we suggest solutions to break these barriers. 


## Introduction {.page_break_before}

A major goal of biomedical research is understanding how changes in biomolecule compositions of cells and tissues lead to disease phenotypes. The genome serves as a library of possible transcripts, some of which are instructions for proteins, and proteins act on metabolites. The genome thus indirectly determines the set of possible cellular phenotypic states, and the exact cellular state at any given time depends on interaction between the genome with endogenous and exogenous environmental cues. To understand how the genome connects to phenotype, measurement of the genome must be accompanied by measurement of downstream layers of the central dogma of biochemistry (Figure 1?). 

Most RNAs encode for proteins, and aside from noncoding RNA and ribozymes, most of the potential functions encoded in a genome are carried out by proteins. Although extensive research has nearly commoditized RNA measurment or transcriptomics [cite transcriptomics review], multiple studies have found poor correlation between proteins and their corresponding mRNA (add citations especially work from christine vogel, maybe add figure here showing poor correlation of CPTAC mRNA/protein data?). Therefore, functional understanding of a biological state demands quantitations of all proteins that are present. Large scale study of the proteins in a biological system is known as proteomics. Proteomics can include 

Many argue that quantities of proteins is not enough. Protein functions are regulated in many ways, including allosteric feedback by metabolites and post-translational modifications such as phosphorylation, acylation, or proteolysis (cite something). A large proportion (what proportion are predicted to be metabolic?) of cellular proteins act on metabolites that are intermediates for cell proliferation or catabolism and energy production. The large scale measurement of the cellular metabolite pools is known as metabolomics, which is sometimes further separated into polar metabolomics and lipidomics. 

Measurement of multiple omic layers is know as multiomics. Mass spectrometry can measure peptides (or proteins, cite top down review) for proteomic analysis, and also metabolites for metabolomic analysis. Over the last decade we have seen an increase in measuring both proteomes and metabolomes from the same sample. Although multiomics can refer to measuring at least any two omes, throughout this review we focus on multiomic measurements between proteomes and metabolomes. 

Integrating data from multiomics should provide more information about the cellular state than the sum of each dataset. Multiomic data integration methods have been developed with different classes of goals. More work is needed to develop methods that take advantage of multiomic data to discover new biological insights into how systems work. 

In this review we cover the following topics related to multiomics:

* sample prepartion methods
* data collection methods
* data integration
* applications to model organisms
* applications to clinical studies

6. Other reviews
    
    Multiomic studies in mitochondria [@url:https://www.sciencedirect.com/science/article/pii/S0165993619303668]
    * discussion of how to prepare samples, QC, and methods to analyze the samples by MS
    * includes mention of linking to functional (phenotype) readout

    Multi-omics approaches to disease [@url:https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1215-1]
    * overview of each omic technology
    * first section is discusses considerations for before multiomic studies: consider the exact disease, sample size, human samples versus model organisms, plan for analysis strategy before collecting data
    * second section is focus on methods for omic integration: 
    * third is future directions: 
    
    
    
    



List of Planned Figures:
1. overview of how omic layers are related showing different 'flavors' of each omic analysis
    * genomics: transcription factor binding (chip-seq), long range structure (ATAC seq?), etc
    * transcriptomics: microarrays, RNA-seq, long read nanopore or pacbio
    * proteomics: interactomics, structural proteomics (thermal proteome profiling or CETSA), PTMomics
    * metabolomics: polar metabolomics, lipidomics, 
   

2. 



## Sample Preparation for Multi-Omic Analysis

sample preparation for intergated multi-omics is a key factor for acquiring repeatable and robust results. 


Integrative multi-omics analysis is a powerful approach to study complex biological responses and has gained popularity in recent years. 

1, Sample preparation for proteomics 



2, Sample preparation for metabolomics

2.1 non-targeted metabolomics 

[@url:https://www.nature.com/articles/s41596-020-0341-5]

2.2 targeted metabolomics

2.3 lipidomics

[@url:https://www.nature.com/articles/nprot.2016.156]

3, Integrative sample prepatation for multi-omics 

In the context of multi-omics analyses, being able to perform multiple measurements on the same sample can also decrease experimental variation. In this section, 
we will review several integrative sample preparation technical advances that increased the capabilities of multiomic analysis. One important goal is to decrease the variation between different samples, ultimately to  . 

 
 [@url:https://pubs.rsc.org/en/content/articlelanding/2020/an/d0an01149e/unauth]
 
 [@url:https://journals.asm.org/doi/10.1128/mSystems.00043-16]
 
 [@url:https://doi.org/10.3389/fgene.2021.635971]
 

 


## New developments of mass spectrometry-based methods for multi-omics 

4.1 proteomics 
 
4.1.1 Traditional standard methods for proteomics 

(Remember to mention here)



4.1.2 Direct infusion methods for proteome analysis (high-throughput methods)

For current proteomic analysis methods, time-consuming chromatographic separation (typically requiring 30–60 min per sample or even longer) is required to protect the coverage, 
repeatiability, robustness and quantification ability. However, with the rapid application of multiomics results in drug development, biomarker discovery studies and clinical 
diagnosis. High-throughput methods is highly desirable to boost these fields forward.[@url:https://doi.org/10.1038/s41592-020-00999-z]

(as a high-throughput method, MALDI based proteome analysis should be mentioned here, for example,  the application of MALDI for identification of species of bacteria and fungas
through their specific peptides)


4.2 metabolomics 

To accurately and reliably interpret data derived from metabolomics and lipidomics studies, enormous mass spectrometry based methods were developed during the past decades.
(remember to mention the application of MALDI for metabolites analysis, although the drawback of MALDI-tof is obvious.(

Drawbacks: 1,the background of organic matrix in the low molecular weight region
           2,the obtained information of MALDI is still very limited, no more than 300 identified metabolites, and also quantification is difficult.   
           3,as a non-consistent ion source, currently TOF is the typical mass analyser for MALDI, which still suffers from relative low resolution. FTICR can connect MALDI )  


4.2.2 Direct infusion mass spectrometry methods for high-throughput analysis of metabolites.()

(direct infusion and so called flow injection MS. do not know the differences, seems saying the same thing.)

[@url:https://www.nature.com/articles/nprot.2016.156]

[@url:https://www.nature.com/articles/nprot.2016.156]

4.4 integrated methods

Mass spectrometry (MS) serves as the centerpiece technology for proteome and metabolome analysis. To gain a better understanding of the multifaceted
networks of myriad actions in complex organisms, integration of different multiomic layers is increasingly explored such as joint methods of different omics. 


[@url:https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c05209]

[@url:https://pubs.acs.org/doi/10.1021/acs.analchem.0c04764?ref=pdf]

[@url:https://pubs.acs.org/doi/10.1021/acs.analchem.0c05209?ref=pdf]





## Multi-Omic Data Integration




  Argonaut data integration manuscript - [@URL:https://doi.org/10.1016/j.patter.2020.100122]

  "Integration strategies of multi-omics data for machine learning analysis" [@PMID:34285775]

  "Multi-omics data integration considerations and study design for biological systems and disease" [@PMID:33347526]


## Mass Spectrometry-based Multi-Omics Applied to Model Organisms

"Multiomics Method Enabled by Sequential Metabolomics and Proteomics for Human Pluripotent Stem-Cell-Derived Cardiomyocytes" [@url:https://pubs.acs.org/doi/full/10.1021/acs.jproteome.1c00611]: 


Multi-omics Reveal Specific Targets of the RNA-Binding Protein Puf3p and Its Orchestration of Mitochondrial Biogenesis [@PMID:29248374]


Mitochondrial protein functions elucidated by multiomic mass spectrometry [@PMID:27669165]


Multi-omic mitoprotease profiling reveals role for oct1p [@PMID:29220658]


"An integrative systems genetic analysis of mammalian lipid metabolism" [@PMID:30814737]
* proteomics and lipidomics of mouse liver across 107 genetically different strains
* 









## Clinical applications of multi-omics (proteomics, metabolomics, and/or lipidomics)

Literature currently focused on proteogenomics (integration of genome, transcriptome, and proteome)"Clinical multi-omics strategies for the effective cancer management"[@URL:https://doi.org/10.3389/fendo.2018.00678]

Holistic approach has the ability to improve prognostics and predictive accuracy of disease phenotypes to aid in better treatment and prevention
"Multi-omics Data Integration, Interpretation, and Its Application"[@URL: https://doi.org/10.1177/1177932219899051

Approaches in cancer research:
- "Integration of Proteomics and Metabolomics Revealed Metabolite–Protein Networks in ACTH-Secreting Pituitary Adenoma" [@URL:https://doi.org/10.3389/fendo.2018.00678]: gas chromatography-mass spectrometry (GC-MS) for metabolomics, plus nano liquid chromatography tandem-mass spectrometry(nanoLC-MS/MS) proteomics. Using metabolomic and proteomic data to identify signaling pathways important in metabolic regulation of tumorigenesis. Reveal biomarkers for disease diagnosis, monitoring and therapeutic targets. Looking for changes in tumor on the pituitary gland (adrenocorticotropic hormone-secreting pituitary adenomas (ACTH-PA)). Downregulated glycolysis and fatty acid synthesis. Myc signaling pathway significantly participated in the metabolic changes and tumorigenesis of ACTH-PA. 
- "Proteometabolomics of bladder cancer: Current and future prospects" [@doi: 10.3233/CBM-150479]: simultaneous proteome and metabolome data from urine and blood for urinary bladder cancer patient surveillance to aid in early detection of bladder cancer


## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
