---
title: 'Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics'
keywords:
- proteomics
- metabolomics
- lipidomics
- mass spectrometry
lang: en-US
date-meta: '2022-01-07'
author-meta:
- Yuming Jiang
- Quinn Dickinson
- Amanda Momenzadeh
- Jesse G. Meyer
header-includes: |-
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta name="citation_title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta property="og:title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta property="twitter:title" content="Mass Spectrometry-based Multi-Omics: Combined Studies of Proteomics, Metabolomics, and/or Lipidomics" />
  <meta name="dc.date" content="2022-01-07" />
  <meta name="citation_publication_date" content="2022-01-07" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Yuming Jiang" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_institution" content="Department of Whatever, University of Something" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="citation_author" content="Quinn Dickinson" />
  <meta name="citation_author_institution" content="Department of Something, University of Whatever" />
  <meta name="citation_author_institution" content="Department of Whatever, University of Something" />
  <meta name="citation_author_orcid" content="XXXX-XXXX-XXXX-XXXX" />
  <meta name="citation_author" content="Amanda Momenzadeh" />
  <meta name="citation_author_institution" content="Department of Biochemistry, Medical College of Wisconsin" />
  <meta name="citation_author_orcid" content="0000-0002-8614-0690" />
  <meta name="citation_author" content="Jesse G. Meyer" />
  <meta name="citation_author_institution" content="Department of Biochemistry, Medical College of Wisconsin" />
  <meta name="citation_author_orcid" content="0000-0003-2753-3926" />
  <meta name="twitter:creator" content="@j_my_sci" />
  <link rel="canonical" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta property="og:url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta property="twitter:url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta name="citation_fulltext_html_url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/" />
  <meta name="citation_pdf_url" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e7cede24b62f2ad3579d25e16dd14f5e68dbf026/" />
  <meta name="manubot_html_url_versioned" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e7cede24b62f2ad3579d25e16dd14f5e68dbf026/" />
  <meta name="manubot_pdf_url_versioned" content="https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e7cede24b62f2ad3579d25e16dd14f5e68dbf026/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
bibliography:
- content/manual-references.json
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
manubot-clear-requests-cache: false
...






<small><em>
This manuscript
([permalink](https://jessegmeyerlab.github.io/2022-multi-omics-review/v/e7cede24b62f2ad3579d25e16dd14f5e68dbf026/))
was automatically generated
from [jessegmeyerlab/2022-multi-omics-review@e7cede2](https://github.com/jessegmeyerlab/2022-multi-omics-review/tree/e7cede24b62f2ad3579d25e16dd14f5e68dbf026)
on January 7, 2022.
</em></small>

## Authors



+ **Yuming Jiang**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [janeroe](https://github.com/janeroe)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
  </small>

+ **Quinn Dickinson**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [janeroe](https://github.com/janeroe)<br>
  <small>
     Department of Something, University of Whatever; Department of Whatever, University of Something
  </small>

+ **Amanda Momenzadeh**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [0000-0002-8614-0690](https://orcid.org/0000-0002-8614-0690)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [amandamomenzadeh](https://github.com/amandamomenzadeh)<br>
  <small>
     Department of Biochemistry, Medical College of Wisconsin
  </small>

+ **Jesse G. Meyer**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon width=16 height=16}
    [0000-0003-2753-3926](https://orcid.org/0000-0003-2753-3926)
    · ![GitHub icon](images/github.svg){.inline_icon width=16 height=16}
    [jessegmeyerlab](https://github.com/jessegmeyerlab)
    · ![Twitter icon](images/twitter.svg){.inline_icon width=16 height=16}
    [j_my_sci](https://twitter.com/j_my_sci)<br>
  <small>
     Department of Biochemistry, Medical College of Wisconsin
     · Funded by Grant R21 AG074234; Grant R35 GM142502
  </small>



## Abstract {.page_break_before}

Studies that integrate unbiased measurements across at least two omics layers are often referred to as "multiomics". Measurable "omes" include the genome, transcriptome, proteome, and metabolome. Any combination of omic measures can be referred to as multiomics; for the scope of this review we focus on research combining proteomics and metabolomics. Mass spectrometry is the leading technique for analysis of the proteome and the metabolome. Due to improvements in sample preparation and data collection, more studies are incorporating both mass spectrometry-based proteomics and metabolomics. In this review, we discuss the perceived value of multiomics, advances in sample preparation and data collection, the current state of multiomic data integration, and clinical examples of multiomic analysis. Finally, we explore major barriers preventing democratization of mass spectrometry based multiomics to the same level as nucleic acid analysis, and we suggest solutions to break these barriers. 


## Introduction {.page_break_before}

A major goal of biomedical research is understanding how changes in biomolecule compositions of cells and tissues lead to disease phenotypes. 
The genome serves as a library of possible transcripts, some of which are instructions for proteins, and proteins act on metabolites. 
The genome thus indirectly determines the set of possible cellular phenotypic states, and the exact cellular state at any given time depends on interaction between the genome with endogenous and exogenous environmental cues. 
To understand how the genome connects to phenotype, measurement of the genome must be accompanied by measurement of downstream layers of the central dogma of biochemistry (Figure 1?). 

Most RNAs encode for proteins, and aside from noncoding RNA and ribozymes, most of the potential functions encoded in a genome are carried out by proteins. 
Although extensive research has nearly commoditized RNA measurment or transcriptomics [@DOI:https://doi.org/10.1038/nrg2484], multiple studies have found poor correlation between proteins and their corresponding mRNA [@DOI:https://doi.org/10.1038/nrg3185].
Therefore, functional understanding of a biological state demands quantitations of all proteins that are present. 

Large scale study of the proteins in a biological system is known as proteomics [@DOI:https://doi.org/10.1038/nature19949]. 
Proteomics originally meant large scale studies of protein quantities [@DOI:10.1038/nprot.2017.040], but it has grown to include large scale studies of protein structure[@DOI:10.1038/nprot.2017.100] [@DOI:10.1586/14789450.2015.1079487] [@DOI:10.1074/jbc.REV119.006218], protein-small molecule interactions[@DOI:10.1146/annurev-pharmtox-052120-013205], and protein-protein interactions [@URL:https://doi.org/10.1016/j.coisb.2017.04.015].. 
Proteomics is increasingly applied to clinical studies to gain understanding of disease mechansims in humans [@PMID:28436239]. 

Many argue that quantities of proteins are not sufficient to enable understanding and dissection of biological systems. 
The presence of a protein does not necessarily equal the presence of that protein's function. 
Protein functions are regulated in many ways, including allosteric feedback by metabolites and post-translational modifications such as phosphorylation, acylation, or proteolysis [@DOI:10.1002/wsbm.1185]. 
Roughly 2,700 cellular proteins are predicted to be enzymes [@DOI:10.1186/gb-2004-6-1-r2], many of which act on metabolites that are intermediates required for cell proliferation or energy production.

The large scale measurement of the cellular metabolite pools is known as metabolomics [@DOI:https://doi.org/10.1039/C9RA08985C], which is sometimes further separated into polar metabolomics and lipidomics [@DOI:https://doi.org/10.1007/s00216-018-1252-y]. 
Due to the chemical heterogeneity of metabolites, metabolomics studies use a variety of different separation and mass spectrometry (MS) techniques, including both gas chromatography (GC) [@PMID:27038389] and liquid chromatograhpy (LC)[@DOI:10.1002/jssc.201100532]. 
The field of metabolomics is trending toward more use of LC-MS over NMR [@DOI:10.1016/j.trac.2018.11.021].

Measurement of multiple omic layers is know as multiomics. 
MS measures peptides (or intact proteins [@DOI:10.1039/C9MO00154A]) for proteomic analysis, but MS also measures metabolites for metabolomic analysis. 
Increasingly studies use MS to measure both proteomes and metabolomes from the same sample. 
Although multiomics can refer to measuring at least any two omes, throughout this review we focus on multiomic measurements between proteomes and metabolomes. 

Integrating data from multiomics should provide more information about the cellular state than the sum of each dataset. 
Multiomic data integration methods have been developed with different classes of goals. 
More work is needed to develop methods that take advantage of multiomic data to discover new biological insights into how systems work. 

Previous reviews have covered related topics. 
A recent review surveyed multiomic studies in mitochondria [@url:https://www.sciencedirect.com/science/article/pii/S0165993619303668] with a focus on sample preparation, qualitity control, methods to collect MS data, and linking measured molecules to phenotypes of respirometry. 
Another review gives an overview of each omic method, considerations before performing a multiomic method, and multiomic integration [@url:https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1215-1].
Misra *et al.* reviewed multiomics with a focus on multiomic integration tools [@doi:https://doi.org/10.1530/JME-18-0055].  
A metabolomics-centered review was recently published that discusses multiomic integration [@DOI:https://doi.org/10.1016/j.aca.2020.10.038].

In this review we focus on the following topics with a focus on MS-based multiomics studies that combine proteomics and metabolomics:

* sample prepartion methods
* data collection methods
* data integration
* applications to model organisms
* applications to clinical studies


List of Planned Figures:

1. overview of how omic layers are related showing different 'flavors' of each omic analysis

  genomics: transcription factor binding (chip-seq), long range structure (ATAC seq?), etc
  transcriptomics: microarrays, RNA-seq, long read nanopore or pacbio
  proteomics: interactomics, structural proteomics (thermal proteome profiling or CETSA), PTMomics
  metabolomics: polar metabolomics, lipidomics, 

2.  Workflow for typical MS-based multi-omics. 
  from sample preparation, extraction to data collection, data intergration and processing.

3. Table of clinical multiomics studies
4. Figure showing concepts in multiomics integration?
5. 



## Sample Preparation for Multi-Omic Analysis

The basics: As with any methodology, the very first but also a crucial step for analytical success is proper sample preparation. This is particularly vital for mass spectrometry-based multi-omic analysis, the selected sample preparation strategy is a key determinant for information that will be obtained. Given the great diversity and complexity of biological samples being tested in proteomics and metabolomics, even subtle differences in sample preparation methods can have profound effects on the types of molecules being extracted, which further affect the retention time, signal stability and ionization efficiency. For example, analysis of different types of biomolecules requires specific sample extraction procedures. The most efficient extraction protocols often only cover a restricted type of biomolecules due to their different physicochemical properties. 
After any necessary dissection or collection, biological tissue samples require a step to 

1, Sample preparation for proteomics 

Protein preparation for MS analysis can be accomplished by many methods, so it is important to understand the steps leading to analysis. While intact proteins are typically studied by gel electrophoresis, the most common mass spectrometry workflows for complex protein samples analyze peptides, which are easier than proteins to fractionate by LC. Peptides also ionize and fragment more efficiently than whole proteins, resulting in spectra that are easier to interpret for protein identification. Peptide preparation involves reduction and alkylation of cysteines, digestion of the sample into peptides, desalting and concentration of the peptides and final analysis of these peptides by ionization (e.g., ESI) plus orbitrap-based MS.




2, Sample preparation for metabolomics

2.1 non-targeted metabolomics 

[@url:https://www.nature.com/articles/s41596-020-0341-5]

2.2 targeted metabolomics

2.3 lipidomics

[@url:https://www.nature.com/articles/nprot.2016.156]

3, Integrative sample prepatation for multi-omics 

In the context of multi-omics analyses, being able to perform multiple measurements on the same sample can also decrease experimental variation. In this section, 
we will review several integrative sample preparation technical advances that increased the capabilities of multiomic analysis. One important goal is to decrease the variation between different samples, ultimately to  . 

 
 [@url:https://pubs.rsc.org/en/content/articlelanding/2020/an/d0an01149e/unauth]
 
 [@url:https://doi.org/10.3389/fgene.2021.635971]
 
 
 -"An Integrated, High-Throughput Strategy for Multiomic Systems Level Analysis"  [@doi:10.1021/acs.jproteome.8b00302]
 
 -"Detergent-Free Simultaneous Sample Preparation Method for Proteomics and Metabolomics" [@doi:https://doi.org/10.1021/acs.jproteome.9b00662]
 
 -"MPLEx: a Robust and Universal Protocol for Single-Sample Integrative Proteomic, Metabolomic, and Lipidomic Analyses" [@doi:https://doi.org/10.1128/mSystems.00043-16]
 
 -"Single-platform ‘multi-omic’ profiling: unified mass spectrometry and computational workflows for integrative proteomics–metabolomics analysis" [@doi:10.1039/C8MO00136G]
 
 -"Three-in-One Simultaneous Extraction of Proteins, Metabolites and Lipids for Multi-Omics" [@doi:https://doi.org/10.3389/fgene.2021.635971] 
 
 -"
 
 -
 
 
 
 

 


## New developments of mass spectrometry-based methods for multi-omics 

4.1 proteomics 
 
4.1.1 Traditional standard methods for proteomics 

(Remember to mention here)



4.1.2 Direct infusion methods for proteome analysis (high-throughput methods)

For current proteomic analysis methods, time-consuming chromatographic separation (typically requiring 30–60 min per sample or even longer) is required to protect the coverage, 
repeatiability, robustness and quantification ability. However, with the rapid application of multiomics results in drug development, biomarker discovery studies and clinical 
diagnosis. High-throughput methods is highly desirable to boost these fields forward.[@url:https://doi.org/10.1038/s41592-020-00999-z]

(as a high-throughput method, MALDI based proteome analysis should be mentioned here, for example,  the application of MALDI for identification of species of bacteria and fungas
through their specific peptides)


4.2 metabolomics 

To accurately and reliably interpret data derived from metabolomics and lipidomics studies, enormous mass spectrometry based methods were developed during the past decades.
(remember to mention the application of MALDI for metabolites analysis, although the drawback of MALDI-tof is obvious.(

Drawbacks: 1,the background of organic matrix in the low molecular weight region
           2,the obtained information of MALDI is still very limited, no more than 300 identified metabolites, and also quantification is difficult.   
           3,as a non-consistent ion source, currently TOF is the typical mass analyser for MALDI, which still suffers from relative low resolution. FTICR can connect MALDI )  


4.2.2 Direct infusion mass spectrometry methods for high-throughput analysis of metabolites.()

(direct infusion and so called flow injection MS. do not know the differences, seems saying the same thing.)
In typical metabolomic analysis workflow, MS is typically coupled with liquid or gas chromatography, enables separating metabolites within a complex sample and further increases the overall sensitivity by minimizing the ion suppression effect. However, this approach also bring for metabolomics analysis is that chromatographic separation is time-consuming (20–60 min per sample), preventing its application for high-throughput metabolomic screens of large scale samples that required for biomarker discovery studies and clinial analysis. 


[@url:https://www.nature.com/articles/nprot.2016.156]

[@url:https://www.nature.com/articles/nprot.2016.156]

4.4 Integrated Strategy for Mass Spectrometry-Based Multiomics

Mass spectrometry (MS) serves as the centerpiece technology for proteome and metabolome analysis. To gain a better understanding of the multifaceted
networks of myriad actions in complex organisms, integration of different multiomic layers is increasingly explored such as joint methods of different omics.


"Multiomics Method Enabled by Sequential Metabolomics and Proteomics for Human Pluripotent Stem-Cell-Derived Cardiomyocytes"  [@doi:]




[@url:https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c05209]

[@url:https://pubs.acs.org/doi/10.1021/acs.analchem.0c04764?ref=pdf]

[@url:https://pubs.acs.org/doi/10.1021/acs.analchem.0c05209?ref=pdf]





## Multi-Omic Data Integration




  Argonaut data integration manuscript - [@URL:https://doi.org/10.1016/j.patter.2020.100122]
  
Multi-omics data integration considerations and study design for biological systems and disease -  [@URL: https://doi.org/10.1039/D0MO00041H]

  List of software available for multiomic (proteomic/metabolomic) integration
    
    Metaboanalyst 4.0 - Metabolomics data analysis, interpretation, and integration with other omics data [@PMID: 31756036]
  
    Paintomics 3.0 (web based) - Joint visualization of transcriptomics and metabolomics data [@URL: https://doi.org/10.1093/bioinformatics/btq594]
    
    integrOmics (R package) - Integrative analysis of two types of omics datasets [@URL: https://doi.org/10.1093/bioinformatics/btp515]
    
    Omics Integrator - Maps protein data to other data sets [@URL: https://doi.org/10.1371/journal.pcbi.1004879]
    
    mixOmics (R package) - Data exploration, dimension reduction, and visualization [@URL: https://doi.org/10.1371/journal.pcbi.1005752]
  

  "Integration strategies of multi-omics data for machine learning analysis" [@PMID:34285775]
  
  Split strategies into five categories
  
     Early integration: Place all omics datasets into single matrix
     
        tends to focus on largest dataset, missing data from other omic datasets [@PMID:18178960] 
        
        simple, easy implementation, allows ML models to learn interactions between layers
        
     Mixed integration: Map each dataset into new representation befrooe combining
     
     Intermediate integration: Transform datasets into common and omic-specific representations
     
     Late Integration: analyze each omic layer separately and combine final predictions
     
     hierachical integration: base integration of datasets on regulatory relationships between omic layers

  "Multi-omics data integration considerations and study design for biological systems and disease" [@PMID:33347526]
  
  mixomics R tools [@URL:https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005752]
  
  
  
  "Multi-omics Data Integration, Interpretation, and Its Application"  [@doi:https://doi.org/10.1177/1177932219899051]
  
  
  Correlation analysis:
  Network Analyses and Data Integration of Proteomics and Metabolomics From Leaves of Two Contrasting Varieties of Sugarcane in Response to Drought [@URL: https://internal-journal.frontiersin.org/articles/10.3389/fpls.2019.01524/full] Regularized Canonical Correlation Analysis
  


## Mass Spectrometry-based Multi-Omics Applied to Model Organisms

"Multiomics Method Enabled by Sequential Metabolomics and Proteomics for Human Pluripotent Stem-Cell-Derived Cardiomyocytes" [@url:https://pubs.acs.org/doi/full/10.1021/acs.jproteome.1c00611]: 


Multi-omics Reveal Specific Targets of the RNA-Binding Protein Puf3p and Its Orchestration of Mitochondrial Biogenesis [@PMID:29248374]


Mitochondrial protein functions elucidated by multiomic mass spectrometry [@PMID:27669165]


Multi-omic mitoprotease profiling reveals role for oct1p [@PMID:29220658]


"An integrative systems genetic analysis of mammalian lipid metabolism" [@PMID:30814737]
* proteomics and lipidomics of mouse liver across 107 genetically different strains
* 









## Clinical applications of multi-omics (proteomics, metabolomics, and/or lipidomics)

Literature currently focused on proteogenomics (integration of genome, transcriptome, and proteome)"Clinical multi-omics strategies for the effective cancer management"[@url:https://doi.org/10.3389/fendo.2018.00678]

Holistic approach has the ability to improve prognostics and predictive accuracy of disease phenotypes to aid in better treatment and prevention
"Multi-omics Data Integration, Interpretation, and Its Application"[@url:https://doi.org/10.1177/1177932219899051]

## Approaches in cancer research:
* "Integration of Proteomics and Metabolomics Revealed Metabolite–Protein Networks in ACTH-Secreting Pituitary Adenoma" [@url:https://doi.org/10.3389/fendo.2018.00678]: gas chromatography-mass spectrometry (GC-MS) for metabolomics, plus nano liquid chromatography tandem-mass spectrometry(nanoLC-MS/MS) proteomics. Using metabolomic and proteomic data to identify signaling pathways important in metabolic regulation of tumorigenesis. Reveal biomarkers for disease diagnosis, monitoring and therapeutic targets. Looking for changes in tumor on the pituitary gland (adrenocorticotropic hormone-secreting pituitary adenomas (ACTH-PA)). Downregulated glycolysis and fatty acid synthesis. Myc signaling pathway significantly participated in the metabolic changes and tumorigenesis of ACTH-PA. 
* "L-Arginine Modulates T Cell Metabolism and Enhances Survival and Anti-tumor Activity"  [@doi:http://dx.doi.org/10.1016/j.cell.2016.09.031]: investigation of proteomic and metabolic adaptations underlying CD4+ T cell activation using MS. Proteins were analyzed by liquid chromatography-coupled mass spectrometry (LC-MS). In parallel, polar metabolites were analyzed by non-targeted flow-injection metabolomics (semi-quantitative method that allows rapid and deep profiling of metabolites. limitations: isobaric compounds cannot be discriminated + possible in-source degradation). Findings: increased L-arginine levels has effects on T cell activation, differentiation, and function.
* "Concurrent lipidomics and proteomics on malignant plasma cells from multiple myeloma patients: Probing the lipid metabolome" [@url:https://doi.org/10.1371/journal.pone.0227455] comparing targeted and untargeted lipidomics and proteomics between relapsed and newly diagnosed MM patients
* "Proteometabolomics of bladder cancer: Current and future prospects" [@doi:10.3233/CBM-150479]: simultaneous proteome and metabolome data from urine and blood for urinary bladder cancer patient surveillance to aid in early detection of bladder cancer 
* "Grade-Dependent Metabolic Reprogramming in Kidney Cancer Revealed by Combined Proteomics and Metabolomics Analysis"   [@doi:10.1158/0008-5472.CAN-14-1703]

## Approaches in COVID-19 research:
* "Proteomic and Metabolomic Characterization of COVID-19 Patient Sera" [@url:https://doi.org/10.1016/j.cell.2020.05.032]: proteomic and metabolomic profiling of serum from 46 COVID-19 patients. Stable isotope labeled proteomics TMTpro (16plex) and and ultra performance liquid chromatography/tandem mass spectrometry (UPLC-MS/MS) untargeted metabolomics to analyze samples. Dysregulation of macrophage, platelet degranulation, complement system, and metabolic suppression in severe COVID-19 patients, useful for selection of biomarkers for severity evaluation. Implemented machine learning model using expression levels of 22 serum proteins and 7 metabolites with overall accuracy of 93.5% in the training set.
*Large-Scale Multi-omic Analysis of COVID-19 Severity: [@url:https://doi.org/10.1016/j.cels.2020.10.003]: investigation of proteins, lipids, and metabolites associated with COVID-19 status and severity in pts admitted with moderate to severe respiratory issues presumably related to SARS-CoV-2 infection.
* "Metabolomic/lipidomic profiling of COVID-19 and individual response to tocilizumab" [@url:https://doi.org/10.1371/journal.ppat.1009243]
* "Multi-Omics Resolves a Sharp Disease-State Shift between Mild and Moderate COVID-19" [@url:https://doi.org/10.1016/j.cell.2020.10.037]
* "Multi-Omics integration analysis of respiratory specimen characterizes baseline molecular determinants associated with COVID-19 diagnosis" [@url:https://doi.org/10.1101/2020.07.06.20147082]
* "Longitudinal multi-omics analysis identifies responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19 trajectories" [@url:https://doi.org/10.1101/2020.09.11.20187369]

## Misc disease states:
* "Integration of metabolomics and proteomics in multiple sclerosis: From biomarkers discovery to personalized medicine" [@url:https://doi.org/10.1002/prca.201500083]: majority of biomarkers for MS are not sensitive and/or specific enough to be used for population screening. From the 188 proposed biomarker candidates for MS found in CSF, only 10 (5%) have been successfully verified, while 20 have been falsified. This is likely due to the inter-individual variation. Integrating omics approaches may help with this pitfall; proteomics in combination with the other “omics” may synergize to proide a more precise spectrum of information capable of better characterizing a phenotype. When proteins alone, metabolites alone and proteins and metabolites together were uploaded into the ingenuity pathway analysis (IPA) tool,  number of matched molecules for “proteins and metabolites” is higher than the sum of molecules matched in the separate searches.
* "High-throughput mediation analysis of human proteome and metabolome identifies mediators of post-bariatric surgical diabetes control"[@url:https://doi.org/10.1038/s41467-021-27289-2] patients with type 2 diabetes  randomized to RYGB vs nonsurgical diabetes/weight management; fasting plasma proteome and metabolome were assayed up to 3 years. Plasma proteome profiling was performed using the high-throughput DNA aptamer-based SOMAscan assay platform. Plasma metabolomics were profiled using a commercial semi-quantitative mass spectrometry-based platform (Metabolon, Inc.)  using UPLC-MS/MS. Main mediator of improved glycemic control was GHR, which was reduced 3 months after RYGB. GH signaling = diabetogenic effects.







## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
