## Introduction {.page_break_before}

A major goal of biomedical research is understanding how changes in biomolecule compositions of cells and tissues lead to disease phenotypes. The genome serves as a library of possible transcripts, some of which are instructions for proteins, and proteins act on metabolites. The genome thus indirectly determines the set of possible cellular phenotypic states, and the exact cellular state at any given time depends on interaction between the genome with endogenous and exogenous environmental cues. To understand how the genome connects to phenotype, measurement of the genome must be accompanied by measurement of downstream layers of the central dogma of biochemistry (Figure 1?). 

Most RNAs encode for proteins, and aside from noncoding RNA and ribozymes, most of the potential functions encoded in a genome are carried out by proteins. Although extensive research has nearly commoditized RNA measurment or transcriptomics [cite transcriptomics review], multiple studies have found poor correlation between proteins and their corresponding mRNA (add citations especially work from christine vogel, maybe add figure here showing poor correlation of CPTAC mRNA/protein data?). Therefore, functional understanding of a biological state demands quantitations of all proteins that are present. Large scale study of the proteins in a biological system is known as proteomics. Proteomics can include 

Many argue that quantities of proteins is not enough. The presence of a protein does not necessarily equal the presence of that protein's function. Protein functions are regulated in many ways, including allosteric feedback by metabolites and post-translational modifications such as phosphorylation, acylation, or proteolysis (cite something). A large proportion (what proportion are predicted to be metabolic?) of cellular proteins act on metabolites that are intermediates for cell proliferation or catabolism and energy production. The large scale measurement of the cellular metabolite pools is known as metabolomics, which is sometimes further separated into polar metabolomics and lipidomics. 

Measurement of multiple omic layers is know as multiomics. Mass spectrometry can measure peptides (or proteins, cite top down review) for proteomic analysis, and also metabolites for metabolomic analysis. Over the last decade we have seen an increase in measuring both proteomes and metabolomes from the same sample. Although multiomics can refer to measuring at least any two omes, throughout this review we focus on multiomic measurements between proteomes and metabolomes. 

Integrating data from multiomics should provide more information about the cellular state than the sum of each dataset. Multiomic data integration methods have been developed with different classes of goals. More work is needed to develop methods that take advantage of multiomic data to discover new biological insights into how systems work. 

In this review we cover the following topics related to multiomics:

* sample prepartion methods
* data collection methods
* data integration
* applications to model organisms
* applications to clinical studies

6. Other reviews
    
    Multiomic studies in mitochondria [@url:https://www.sciencedirect.com/science/article/pii/S0165993619303668]
    * discussion of how to prepare samples, QC, and methods to analyze the samples by MS
    * includes mention of linking to functional (phenotype) readout

    Multi-omics approaches to disease [@url:https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1215-1]
    * overview of each omic technology
    * first section is discusses considerations for before multiomic studies: consider the exact disease, sample size, human samples versus model organisms, plan for analysis strategy before collecting data
    * second section is focus on methods for omic integration: 
    * third is future directions: 
    
    Integrated omics: tools, advances and future approaches[@doi:https://doi.org/10.1530/JME-18-0055]
    
   

List of Planned Figures:
1. overview of how omic layers are related showing different 'flavors' of each omic analysis
    * genomics: transcription factor binding (chip-seq), long range structure (ATAC seq?), etc
    * transcriptomics: microarrays, RNA-seq, long read nanopore or pacbio
    * proteomics: interactomics, structural proteomics (thermal proteome profiling or CETSA), PTMomics
    * metabolomics: polar metabolomics, lipidomics, 
   

2.  Workflow for typical MS-based multi-omics. 
   * from sample preparation, extraction to data collection, data intergration and processing.

