## Multi-Omic Data Integration

Using mass spectrometry, the data collected is proteomics and metabolomics. Integration of these datase

Reviews:

Multi-omics Data Integration, Interpretation, and Its Application [@URL:https://journals.sagepub.com/doi/10.1177/1177932219899051]

Network approaches to systems biology analysis of complex disease: integrative methods for multi-omics data [@URL:https://academic.oup.com/bib/article/19/6/1370/3904446]

  Argonaut data integration manuscript - [@URL:https://doi.org/10.1016/j.patter.2020.100122]
  
Multi-omics data integration considerations and study design for biological systems and disease -  [@URL: https://doi.org/10.1039/D0MO00041H]

  List of software available for multiomic (proteomic/metabolomic) integration
    
    Metaboanalyst 4.0 - Metabolomics data analysis, interpretation, and integration with other omics data [@PMID: 31756036]
  
    Paintomics 3.0 (web based) - Joint visualization of transcriptomics and metabolomics data [@URL: https://doi.org/10.1093/bioinformatics/btq594]
    
    integrOmics (R package) - Integrative analysis of two types of omics datasets [@URL: https://doi.org/10.1093/bioinformatics/btp515]
    
    Omics Integrator - Maps protein data to other data sets [@URL: https://doi.org/10.1371/journal.pcbi.1004879]
    
    mixOmics (R package) - Data exploration, dimension reduction, and visualization [@URL: https://doi.org/10.1371/journal.pcbi.1005752]
  

  "Integration strategies of multi-omics data for machine learning analysis" [@PMID:34285775]
  
  Split strategies into five categories
  
     Early integration: Place all omics datasets into single matrix
     
        tends to focus on largest dataset, missing data from other omic datasets [@PMID:18178960] 
        
        simple, easy implementation, allows ML models to learn interactions between layers
        
     Mixed integration: Map each dataset into new representation before combining
     
     Intermediate integration: Transform datasets into common and omic-specific representations
     
     Late Integration: analyze each omic layer separately and combine final predictions
     
     hierachical integration: base integration of datasets on regulatory relationships between omic layers

  "Multi-omics data integration considerations and study design for biological systems and disease" [@PMID:33347526]
  
  mixomics R tools [@URL:https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005752]
  
  
  
  "Multi-omics Data Integration, Interpretation, and Its Application"  [@doi:https://doi.org/10.1177/1177932219899051]
  
  
  Correlation analysis:
  Network Analyses and Data Integration of Proteomics and Metabolomics From Leaves of Two Contrasting Varieties of Sugarcane in Response to Drought [@URL: https://internal-journal.frontiersin.org/articles/10.3389/fpls.2019.01524/full] Regularized Canonical Correlation Analysis
  
  
 Clustering and variable selection evaluation of 13 unsupervised methods for multi-omics data integration [@URL:https://academic.oup.com/bib/article/21/6/2011/5645549?login=true]


Data integration and predictive modeling methods for multi-omics datasets [@URL: https://pubs.rsc.org/en/content/articlehtml/2018/mo/c7mo00051k]


Specifically proteomics and metabolomics:
[@URL:https://www.nature.com/articles/s41598-018-25035-1#MOESM1] Uses feature selection for proteomic and metabolomic data to create logistic regression predicting survivors from non-survivors during severe septic shock

In Cambiaghi et al 2018 [@URL:https://www.nature.com/articles/s41598-018-25035-1#MOESM1], targeted metabolomics and untarged proteomics were collected from septic shock patients from day one to day seven. To integrate the data, logistic regression models were built to link the metabolomics and proteomics data with patient outcome. Feature selection was performed on the metabolomic data set with minimal-redundancy-maximal-relevance (mRMR) [PMID:16119262]. Further feature selection was performed using an elastic net logistic model to select variables up to the sample size of 17.  Linear Discriminant analysis and partial least squares discriminant analysis were used on the top scoring metabolites. To integrate targeted metabolomics with untargeted proteomics, mRMR was applied. The top 50 ranked proteins were combined with top 50 ranked metabolites and mRMR repeated. Additionally, clinical data was integrated in a similar manner. This method represents an early integration strategy with feature selection as the means to determine biological relationships.

[@URL:https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC2556022/] Integration of Metabolomic and Proteomic Phenotypes - correlation and protein metabolite covariance
In Wienkoop et al 2008


[@URL:https://www.frontiersin.org/articles/10.3389/fendo.2018.00678/full] Integration of Proteomics and Metabolomics Revealed Metaboliteâ€“Protein Networks in ACTH-Secreting Pituitary Adenoma - hierarchical clustering, pathway enrichment analysis
In Feng et al 2018, metabolomics data and proteomics data were collected on adrenocorticotropic hormone-secreting pituitary adenomas. Metabolomics data was prepared and analyzed with MetaboAnalyst [@URL:http://www.metaboanalyst.ca/faces/home.xhtml]. Heirarchical cluster analysis was performed with MetabolAnalyst. Proteomics data was analyzed with ProteinPilot and searched against the SwissProt Database. Protein enrichment analysis was perfomred with Gene Set Enrichment Analysis [@URL:http://software.broadinstitute.org/gsea/msigdb/index.jsp]. Further integration such as network analysis was performed in MetaboAnalyst. This represents a late integration strategy where each dataset is processed separately and conclusions drawn from the connections linking the separate analyses.

[@URL:https://www.dovepress.com/integrated-datasets-of-proteomic-and-metabolomic-biomarkers-to-predict-peer-reviewed-fulltext-article-DMSO] - Integrated Datasets of Proteomic and Metabolomic Biomarkers to Predict Its Impacts on Comorbidities of Type 2 Diabetes Mellitus - "ingenuity pathway analysis"
In Cheema et al 2020, 

[@URL:https://pubs.acs.org/doi/10.1021/acs.jafc.0c00986] - Integrated Proteomics and Metabolomics Analysis Highlights Correlative Metabolite-Protein Networks in Soybean Seeds Subjected to Warm-Water Soaking - hierarchical clustering of differentially expressed proteins - "MapMan" - GO enrichment
In Min et al 2020, proteomics and metabolomics data was collected on soybean seeds soaked at 4, 25, and 55C. Proteomics data was processed using MaxQuant and Perseus. Enrichment of significantly changed proteins was analyzed using MapMan[cite]. Metabolomic data was analysed by unit-varience scaling the data and using principal component analysis and partial least-squares discriminant analysis using Soft Independent Modeling of Class Analogy (SIMCA)[cite]. MetaboAnalyst was used for Hierachical Cluster analysis between samples and compounds. Pearson's correlation analysis was perfomed with Statistical Analysis System (SAS)[cite]. This represents a late integration strategy where each dataset is processed separately and conclusions drawn from the connections linking the separate analyses.




How are people analyzing data? What software is available for combinational analysis? Why aren't these being utilized? What gaps in analysis are there in the software?

Metabolomics and Proteomics represent two facets of the spectrum of multiomic data analysis. Through the integration of these datasets, insights can be gleaned that may not be evident from each individual source. The strategies for the integration of any set of multiomic data can be broadly divided into two categories, namely early integration and late integration[source]. Early integration strategies combine the datasets, potentially with adjustments to make the datasets compatably scaled, before further analysis [Source]. The essence of late integration strategies is to analyze each dataset separately then combine knowledge gained from each to support further conclusions [Source]. Each strategy employed have positive and negative aspects that affect the downsteam analysis of the multiomic data. The



